In [90]:
from requests_html import HTMLSession

In [91]:
session = HTMLSession()

In [105]:
url = 'http://www.sse.com.cn/assortment/stock/list/info/executives/index.shtml?COMPANY_CODE=600004'

In [93]:
r = session.get(url)

In [94]:
print(r.html.text)

高管人员 | 上海证券交易所
var col_id=9067 ; var col_old_id="";
var siteId = 28; var tableFun = {}; var tableData = {}; var searchUrl = "/home/search/"; var bizUrl = "http://biz.sse.com.cn/"; var staticUrl = "http://static.sse.com.cn"; var sseQueryURL = "http://query.sse.com.cn/"; var hq_queryUrl = "http://yunhq.sse.com.cn:32041"; //var pdfUrl = "http://static.sse.com.cn"; //var downloadUrl = "http://query.sse.com.cn"; //var sseTestQueryURL = "http://query.sse.com.cn/"; //var siteUrl = "http://www.sse.com.cn/";
热线电话： 400-8888-400 投资者服务中心
APP下载
上交所移动App
微博微信
上交所官方微信
子站链接
投资者服务中心
上证投资者教育网站
上证e互动
上证债券信息网
上证路演中心
业务办理专区
上市公司专区
会员/证券机构专区
衍生品与基金专区
债券专区
CA服务专区
媒体专区
证监局专区
繁体 English
.desksearchTableTr { background-color : #F0F0F0;}
半年报 增持 回购 债券 监管关注 财务报表
Toggle navigation
document.write(sseMenuObj.menu_tab_text);
document.write(sseMenuObj.menu_con_text);
document.write(sseMenuObj.mobile_menu_text);
document.write(sseMenuObj.initBreadMenu());
高管人员
document.write(sseMenuObj.initLeftMenu());
扫一扫下载
"上交所移动App"


In [95]:
r.html.links

{'/',
 '/aboutus/contactus/',
 '/aboutus/cpc/home/',
 '/aboutus/overview/',
 '/aboutus/recruitment/sse/',
 '/aboutus/sseintroduction/events/',
 '/aboutus/sseintroduction/trademark/',
 '/assortment/fund/etf/home/',
 '/assortment/fund/lof/home/',
 '/assortment/options/home/',
 '/assortment/overview/',
 '/disclosure/bond/ratios/',
 '/disclosure/dealinstruc/suspension/',
 '/disclosure/diclosure/public/',
 '/disclosure/listedinfo/periodic/',
 '/disclosure/overview/',
 '/home/',
 '/home/app/',
 '/home/app/update/',
 '/home/feedback/',
 '/home/legal/',
 '/home/links/',
 '/home/map/',
 '/home/weixin/',
 '/lawandrules/guide/disclosure/dailymemo/',
 '/lawandrules/overview/',
 '/lawandrules/regulations/csrcorder/',
 '/lawandrules/rules/law/securities/',
 '/lawandrules/sserules/latest/',
 '/market/bonddata/livelybond/',
 '/market/bonddata/overview/',
 '/market/othersdata/hkexsctrading/',
 '/market/overview/',
 '/market/price/trends/',
 '/market/stockdata/overview/day/',
 '/services/hkexsc/home/',


In [96]:
r.html.absolute_links

{'http://big5.sse.com.cn',
 'http://biz.sse.com.cn/bond/',
 'http://biz.sse.com.cn/fund/',
 'http://biz.sse.com.cn/list/',
 'http://biz.sse.com.cn/media/',
 'http://biz.sse.com.cn/member/',
 'http://bond.sse.com.cn/',
 'http://csrc.sse.com.cn',
 'http://edu.sse.com.cn/',
 'http://english.sse.com.cn',
 'http://mb.sseinfo.com/ComInfoServer/ssegwappdownload.jsp',
 'http://my.sse.com.cn/uc/view/feedback.shtml',
 'http://my.sse.com.cn/uc/view/index.shtml',
 'http://phzq.sseinfo.com/',
 'http://roadshow.sseinfo.com/',
 'http://sns.sseinfo.com/',
 'http://www.miibeian.gov.cn/',
 'http://www.sse.com.cn/',
 'http://www.sse.com.cn/aboutus/contactus/',
 'http://www.sse.com.cn/aboutus/cpc/home/',
 'http://www.sse.com.cn/aboutus/overview/',
 'http://www.sse.com.cn/aboutus/recruitment/sse/',
 'http://www.sse.com.cn/aboutus/sseintroduction/events/',
 'http://www.sse.com.cn/aboutus/sseintroduction/trademark/',
 'http://www.sse.com.cn/assortment/fund/etf/home/',
 'http://www.sse.com.cn/assortment/fund/

In [97]:
sel = '#tableData_ > div.table-responsive.sse_table_T01.tdclickable > table > tbody > tr:nth-child(2) > td:nth-child(2)'

#js_stockName > span

In [98]:
results = r.html.find(sel)

In [99]:
results

[]

In [100]:
results[0].text

IndexError: list index out of range

In [13]:
results[0].absolute_links

{'https://www.jianshu.com/nb/130182'}

In [14]:
list(results[0].absolute_links)[0]

'https://www.jianshu.com/nb/130182'

In [15]:
def get_text_link_from_sel(sel):
    mylist = []
    try:
        results = r.html.find(sel)
        for result in results:
            mytext = result.text
            mylink = list(result.absolute_links)[0]
            mylist.append((mytext, mylink))
        return mylist
    except:
        return None

In [16]:
print(get_text_link_from_sel(sel))

[('玉树芝兰》里，写了一系列', 'https://www.jianshu.com/nb/130182')]


In [17]:
sel = 'body > div.note > div.post > div.article > div.show-content > div > p:nth-child(6) > a'

In [18]:
print(get_text_link_from_sel(sel))

[('如何用Python做词云？》受到了读者们非常热烈的欢迎。', 'https://www.jianshu.com/p/e4b24a734ccc')]


In [19]:
sel = 'body > div.note > div.post > div.article > div.show-content > div > p > a'

In [20]:
print(get_text_link_from_sel(sel))

[('玉树芝兰》里，写了一系列', 'https://www.jianshu.com/nb/130182'), ('如何用Python做词云？》受到了读者们非常热烈的欢迎。', 'https://www.jianshu.com/p/e4b24a734ccc'), ('数据科学相关的文章。', 'https://www.jianshu.com/p/30b4fa6793f6'), ('索引贴。', 'https://www.jianshu.com/p/30b4fa6793f6'), ('教程做完了词云，发现如果对中文文本做分析，就会出现乱码：', 'https://www.jianshu.com/p/e4b24a734ccc'), ('如何安装Python运行环境Anaconda？（视频教程）》，自己尝试安装Anaconda，运行起第一个Jupyter Notebook，输出一个“Hello world!”出来。', 'https://www.jianshu.com/p/772740d57576'), ('如何用Python做词云？》一步步执行。用几行Python代码，你就可以做出这样的词云来。', 'https://www.jianshu.com/p/e4b24a734ccc'), ('如何用Python做词云？（基础篇视频教程）》，供你观看。', 'https://www.jianshu.com/p/a2924231037c'), ('如何用Python做中文分词？》，你就能做出这样的中文词云。', 'https://www.jianshu.com/p/721190534061'), ('Python编程遇问题，文科生怎么办？》的', 'https://www.jianshu.com/p/af6201d2f0c6'), ('如何在Jupyter Notebook中使用Python虚拟环境？》。', 'https://www.jianshu.com/p/afea092dda1d'), ('如何用Python做情感分析？》这篇文章，分别从英文和中文两个案例，分别采用不同的软件包，针对性地解决应用需求。', 'https://www.jianshu.com/p/d50a14541d01'), ('如何用Python做舆情时间序列可视化？》这篇文章，一步步指引你在时间刻度上

In [21]:
import pandas as pd

In [22]:
df = pd.DataFrame(get_text_link_from_sel(sel))

In [23]:
df

0  \
0                                        玉树芝兰》里，写了一系列   
1                        如何用Python做词云？》受到了读者们非常热烈的欢迎。   
2                                          数据科学相关的文章。   
3                                                索引贴。   
4                        教程做完了词云，发现如果对中文文本做分析，就会出现乱码：   
5   如何安装Python运行环境Anaconda？（视频教程）》，自己尝试安装Anaconda，...   
6       如何用Python做词云？》一步步执行。用几行Python代码，你就可以做出这样的词云来。   
7                       如何用Python做词云？（基础篇视频教程）》，供你观看。   
8                      如何用Python做中文分词？》，你就能做出这样的中文词云。   
9                               Python编程遇问题，文科生怎么办？》的   
10                如何在Jupyter Notebook中使用Python虚拟环境？》。   
11  如何用Python做情感分析？》这篇文章，分别从英文和中文两个案例，分别采用不同的软件包，针...   
12   如何用Python做舆情时间序列可视化？》这篇文章，一步步指引你在时间刻度上可视化情感分析结果：   
13                              Python编程遇问题，文科生怎么办？》的   
14                    如何用Python和机器学习训练中文文本情感分类模型？》一文。   
15  如何用Python和R对《权力的游戏》故事情节做情绪分析？》一文分析了《权力的游戏》中某一集...   
16                                         文章，跟结尾做对比。   
17  如何用Python提取中文关键词？》一文。它采用词汇向量化、TextRank等成熟的关键词提...   
18                        文科生如何高效学数据科学？》。文中提到了以下几个方面：   
19    如何高效入门Github？》一文提供了文档和视频教程资源，希望能对你掌握这个数据富矿提供帮助。   
20  如何高效学Python？》帮助你给自己的学习特性做出了分类。根据分类的结果，你可以选择更适合...   
21        贷还是不贷：如何用Python和机器学习帮你决策？》中的案例，选择了贷款审批辅助决策。   
22  Python编程遇问题，文科生怎么办？》的第一部分，对这些问题做了详细的阐述，请根据列出的步...   
23                                        这篇文章展示给你了一种   
24                              如何用Python从海量文本抽取主题？》。   
25  如何用Python和机器学习训练中文文本情感分类模型？》一文中，我不仅对停用词处理方式进行了...   
26       如何用Python和深度神经网络发现即将流失的客户？》为你介绍了深度神经网络的基本结构。   
27  如何用Python和深度神经网络发现即将流失的客户？》一文末尾还为你提供了进一步掌握深度学习...   
28                            Tensorflow执行pip升级安装的坑》。   
29  如何用Python和深度神经网络识别图像？》一文，举例分类哆啦a梦和瓦力这两个机器人的各种花...   
30                          如何用Python和深度神经网络寻找近似图片？》。   
31                                            讲解答疑视频。   
32                                这段视频里，我主要谈及了以下几个方面：   
33                            如何免费云端运行Python深度学习框架？》。   
34  如何用Python读取开放数据？》一文中，我为你讲解了如何把开放数据集下载并且在Python...   
35  如何用R和API免费获取Web数据？》一文，我们使用R读取维基百科API，获得指定条目的访问...   
36       如何用Python爬数据？（一）网页抓取》一文为你介绍了非常人性化、易用的网页抓取软件包   
37                            如何用Python批量提取PDF文本内容？》。   
38  玉树芝兰》专栏目前的数据科学类文章进行了梳理和归类，建立了链接，以帮你看清它们之间的逻辑依赖关系。   
39                               “玉树芝兰”(nkwangshuyi)。   
40                       如何高效入门数据科学？》，里面还有更多的有趣问题及解法。   

                                                    1  
0                   https://www.jianshu.com/nb/130182  
1              https://www.jianshu.com/p/e4b24a734ccc  
2              https://www.jianshu.com/p/30b4fa6793f6  
3              https://www.jianshu.com/p/30b4fa6793f6  
4              https://www.jianshu.com/p/e4b24a734ccc  
5              https://www.jianshu.com/p/772740d57576  
6              https://www.jianshu.com/p/e4b24a734ccc  
7              https://www.jianshu.com/p/a2924231037c  
8              https://www.jianshu.com/p/721190534061  
9              https://www.jianshu.com/p/af6201d2f0c6  
10             https://www.jianshu.com/p/afea092dda1d  
11             https://www.jianshu.com/p/d50a14541d01  
12             https://www.jianshu.com/p/4ea083874df4  
13             https://www.jianshu.com/p/af6201d2f0c6  
14             https://www.jianshu.com/p/29aa3ad63f9d  
15             https://www.jianshu.com/p/0c782715e58a  
16             https://www.jianshu.com/p/0c782715e58a  
17             https://www.jianshu.com/p/cf383fd471bb  
18             https://www.jianshu.com/p/e3f52e0860fd  
19             https://www.jianshu.com/p/13d356e76659  
20             https://www.jianshu.com/p/3678e3dd29f5  
21             https://www.jianshu.com/p/67a71e366516  
22             https://www.jianshu.com/p/af6201d2f0c6  
23             https://www.jianshu.com/p/af6201d2f0c6  
24             https://www.jianshu.com/p/fdde9fc03f94  
25             https://www.jianshu.com/p/29aa3ad63f9d  
26             https://www.jianshu.com/p/7c08e9d79514  
27             https://www.jianshu.com/p/7c08e9d79514  
28             https://www.jiansh

In [24]:
df.columns = ['text', 'link']

In [25]:
df

text  \
0                                        玉树芝兰》里，写了一系列   
1                        如何用Python做词云？》受到了读者们非常热烈的欢迎。   
2                                          数据科学相关的文章。   
3                                                索引贴。   
4                        教程做完了词云，发现如果对中文文本做分析，就会出现乱码：   
5   如何安装Python运行环境Anaconda？（视频教程）》，自己尝试安装Anaconda，...   
6       如何用Python做词云？》一步步执行。用几行Python代码，你就可以做出这样的词云来。   
7                       如何用Python做词云？（基础篇视频教程）》，供你观看。   
8                      如何用Python做中文分词？》，你就能做出这样的中文词云。   
9                               Python编程遇问题，文科生怎么办？》的   
10                如何在Jupyter Notebook中使用Python虚拟环境？》。   
11  如何用Python做情感分析？》这篇文章，分别从英文和中文两个案例，分别采用不同的软件包，针...   
12   如何用Python做舆情时间序列可视化？》这篇文章，一步步指引你在时间刻度上可视化情感分析结果：   
13                              Python编程遇问题，文科生怎么办？》的   
14                    如何用Python和机器学习训练中文文本情感分类模型？》一文。   
15  如何用Python和R对《权力的游戏》故事情节做情绪分析？》一文分析了《权力的游戏》中某一集...   
16                                         文章，跟结尾做对比。   
17  如何用Python提取中文关键词？》一文。它采用词汇向量化、TextRank等成熟的关键词提...   
18                        文科生如何高效学数据科学？》。文中提到了以下几个方面：   
19    如何高效入门Github？》一文提供了文档和视频教程资源，希望能对你掌握这个数据富矿提供帮助。   
20  如何高效学Python？》帮助你给自己的学习特性做出了分类。根据分类的结果，你可以选择更适合...   
21        贷还是不贷：如何用Python和机器学习帮你决策？》中的案例，选择了贷款审批辅助决策。   
22  Python编程遇问题，文科生怎么办？》的第一部分，对这些问题做了详细的阐述，请根据列出的步...   
23                                        这篇文章展示给你了一种   
24                              如何用Python从海量文本抽取主题？》。   
25  如何用Python和机器学习训练中文文本情感分类模型？》一文中，我不仅对停用词处理方式进行了...   
26       如何用Python和深度神经网络发现即将流失的客户？》为你介绍了深度神经网络的基本结构。   
27  如何用Python和深度神经网络发现即将流失的客户？》一文末尾还为你提供了进一步掌握深度学习...   
28                            Tensorflow执行pip升级安装的坑》。   
29  如何用Python和深度神经网络识别图像？》一文，举例分类哆啦a梦和瓦力这两个机器人的各种花...   
30                          如何用Python和深度神经网络寻找近似图片？》。   
31                                            讲解答疑视频。   
32                                这段视频里，我主要谈及了以下几个方面：   
33                            如何免费云端运行Python深度学习框架？》。   
34  如何用Python读取开放数据？》一文中，我为你讲解了如何把开放数据集下载并且在Python...   
35  如何用R和API免费获取Web数据？》一文，我们使用R读取维基百科API，获得指定条目的访问...   
36       如何用Python爬数据？（一）网页抓取》一文为你介绍了非常人性化、易用的网页抓取软件包   
37                            如何用Python批量提取PDF文本内容？》。   
38  玉树芝兰》专栏目前的数据科学类文章进行了梳理和归类，建立了链接，以帮你看清它们之间的逻辑依赖关系。   
39                               “玉树芝兰”(nkwangshuyi)。   
40                       如何高效入门数据科学？》，里面还有更多的有趣问题及解法。   

                                                 link  
0                   https://www.jianshu.com/nb/130182  
1              https://www.jianshu.com/p/e4b24a734ccc  
2              https://www.jianshu.com/p/30b4fa6793f6  
3              https://www.jianshu.com/p/30b4fa6793f6  
4              https://www.jianshu.com/p/e4b24a734ccc  
5              https://www.jianshu.com/p/772740d57576  
6              https://www.jianshu.com/p/e4b24a734ccc  
7              https://www.jianshu.com/p/a2924231037c  
8              https://www.jianshu.com/p/721190534061  
9              https://www.jianshu.com/p/af6201d2f0c6  
10             https://www.jianshu.com/p/afea092dda1d  
11             https://www.jianshu.com/p/d50a14541d01  
12             https://www.jianshu.com/p/4ea083874df4  
13             https://www.jianshu.com/p/af6201d2f0c6  
14             https://www.jianshu.com/p/29aa3ad63f9d  
15             https://www.jianshu.com/p/0c782715e58a  
16             https://www.jianshu.com/p/0c782715e58a  
17             https://www.jianshu.com/p/cf383fd471bb  
18             https://www.jianshu.com/p/e3f52e0860fd  
19             https://www.jianshu.com/p/13d356e76659  
20             https://www.jianshu.com/p/3678e3dd29f5  
21             https://www.jianshu.com/p/67a71e366516  
22             https://www.jianshu.com/p/af6201d2f0c6  
23             https://www.jianshu.com/p/af6201d2f0c6  
24             https://www.jianshu.com/p/fdde9fc03f94  
25             https://www.jianshu.com/p/29aa3ad63f9d  
26             https://www.jianshu.com/p/7c08e9d79514  
27             https://www.jianshu.com/p/7c08e9d79514  
28             https://www.jia

In [26]:
df.to_csv('output.csv', encoding='gbk', index=False)

# Crawler SSE company executives

In [512]:
import requests
import time 
from jsonpath import jsonpath
import json
import time
import numpy as np

In [447]:
url = 'http://query.sse.com.cn/security/stock/getStockListData2.do?&jsonCallBack=jsonpCallback23450&isPagination=true&stockCode=&csrcCode=&areaName=&stockType=1&pageHelp.cacheSize=1&pageHelp.beginPage=1&pageHelp.pageSize=25&pageHelp.pageNo=1&_=1535448575333'


In [448]:
headers = {
    'Cookie':'yfx_c_g_u_id_10000042=_ck18082710051716306753275593750; VISITED_COMPANY_CODE=%5B%22600004%22%2C%22600000%22%5D; VISITED_STOCK_CODE=%5B%22600004%22%2C%22600000%22%5D; seecookie=%5B600004%5D%3A%u767D%u4E91%u673A%u573A%2C%5B600000%5D%3A%u6D66%u53D1%u94F6%u884C; VISITED_MENU=%5B%2211189%22%2C%228765%22%2C%228464%22%2C%228451%22%2C%228523%22%2C%228466%22%2C%229062%22%2C%229066%22%2C%229055%22%2C%229067%22%2C%228528%22%5D; yfx_f_l_v_t_10000042=f_t_1535335516955__r_t_1535419438032__v_t_1535448573927__r_c_1',
    'User-Agent':'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36',
    'Referer':'http://www.sse.com.cn/assortment/stock/list/share/'
}

In [449]:
response=requests.get(url,headers=headers)

In [450]:
response

<Response [200]>

In [567]:
def find_pageA(c): #根据传递参数c（提取的页数）来选择目标url地址
        return 'http://query.sse.com.cn/security/stock/getStockListData2.do?&jsonCallBack=jsonpCallback13897&isPagination=true&stockCode=&csrcCode=&areaName=&stockType=1&pageHelp.cacheSize=1&pageHelp.beginPage='+str(c)+'&pageHelp.pageSize=25&pageHelp.pageNo='+str(c)+'&pageHelp.endPage='+str(c)+'1&_=151731428806'+str(c)
def datascreenA(a):#封装解析输出的部分
    COMPANY_CODE=jsonpath(a,'$..pageHelp..COMPANY_CODE')
    COMPANY_ABBR=jsonpath(a,'$..pageHelp..COMPANY_ABBR')
    totalShares=jsonpath(a,"$..pageHelp..totalShares")
    totalFlowShares=jsonpath(a,'$..pageHelp..totalFlowShares')
    #print('公司/A股代码','\t','公司/A股简称','\t','A股总资本','\t','A股流动资本')

    return COMPANY_CODE
def collect_58_pagesA():#调取30页，相当于主函数
    c=1
    L=[]
    while(c<59):
        #print('第', c, '页:')
        response=requests.get(find_pageA(c),headers=headers)
        a='{"content":'+response.text[19:-1]+'}'
        b=json.loads(a)
#       print(b)
        page_list = datascreenA(b)
        L.append(page_list)
        c+=1
    return L


In [568]:
start_time = time.time()
company_codes = collect_30_pagesA()
cost_time = time.time()-start_time
print(str(cost_time)+'s')

31.940789937973022s


In [569]:
# 2D list to 1D list(flatten)
company_codes= np.concatenate(company_codes)

In [570]:
header_company = {
    'Cookie':'yfx_c_g_u_id_10000042=_ck18082710051716306753275593750; VISITED_COMPANY_CODE=%5B%22600004%22%2C%22600000%22%5D; VISITED_STOCK_CODE=%5B%22600004%22%2C%22600000%22%5D; seecookie=%5B600004%5D%3A%u767D%u4E91%u673A%u573A%2C%5B600000%5D%3A%u6D66%u53D1%u94F6%u884C; VISITED_MENU=%5B%2211189%22%2C%228765%22%2C%228464%22%2C%228451%22%2C%228523%22%2C%228466%22%2C%229062%22%2C%229066%22%2C%229055%22%2C%229067%22%2C%228528%22%5D; yfx_f_l_v_t_10000042=f_t_1535335516955__r_t_1535419438032__v_t_1535448573927__r_c_1',
    'User-Agent':'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36',
    'Referer':'http://www.sse.com.cn/assortment/stock/list/info/executives/index.shtml'
}  

In [571]:
url_executives= 'http://query.sse.com.cn/commonQuery.do?jsonCallBack=jsonpCallback80740&isPagination=true&productid=600020&sqlId=COMMON_SSE_ZQPZ_GG_GGRYLB_L&pageHelp.pageSize=25&pageHelp.pageCount=50&pageHelp.pageNo=1&pageHelp.beginPage=1&pageHelp.cacheSize=1&pageHelp.endPage=5&_=1535506282048'


In [572]:
response_executives=requests.get(url_executives,headers=header_company)
response_executives=response_executives.text[19:-1]
json_executives=json.loads(response_executives)
executives = jsonpath(json_executives,'$..result..NAME')

In [573]:
print(executives)

['王继东', '张付雄', '段永灿', '田燕玲', '刘剑君', '刘剑君', '杨亚子', '杨亚子']


# company information

In [489]:
url_company= 'http://query.sse.com.cn/commonQuery.do?jsonCallBack=jsonpCallback00001&isPagination=false&sqlId=COMMON_SSE_ZQPZ_GP_GPLB_C&productid=600050&_=11111111111111'


In [494]:
response_company=requests.get(url_company,headers=header_company)
print(response_company.text)
response_company=response_company.text[19:-1]
print(response_company)
json_company=json.loads(response_company)
COMPANY_CODE = jsonpath(json_company,'$..result..COMPANY_CODE')
COMPANY_ABBR = jsonpath(json_company,'$..result..COMPANY_ABBR')
FULLNAME = jsonpath(json_company,'$..result..FULLNAME')

jsonpCallback00001({"actionErrors":[],"actionMessages":[],"errorMessages":[],"errors":{},"fieldErrors":{},"isPagination":"false","jsonCallBack":"jsonpCallback00001","locale":"zh_CN","pageHelp":{"beginPage":1,"cacheSize":5,"data":null,"endDate":null,"endPage":null,"objectResult":null,"pageCount":null,"pageNo":1,"pageSize":10,"searchDate":null,"sort":null,"startDate":null,"total":0},"result":[{"ENGLISH_ABBR":"China Unicom","LEGAL_REPRESENTATIVE":"王晓初                        ","REPR_PHONE":"-","CSRC_CODE_DESC":"信息传输、软件和信息技术服务业","E_MAIL_ADDRESS":"ir@chinaunicom.cn","SSE_CODE_DESC":"公用事业","WWW_ADDRESS":"www.chinaunicom-a.com","SECURITY_CODE_A":"600050","SECURITY_CODE_B":"-","COMPANY_CODE":"600050","OFFICE_ADDRESS":"北京市西城区金融大街21号","SECURITY_ABBR_A":"中国联通","COMPANY_ABBR":"中国联通","AREA_NAME_DESC":"上海","STATE_CODE_A_DESC":"上市","CSRC_MIDDLE_CODE_DESC":"-","FULLNAME":"中国联合网络通信股份有限公司","FOREIGN_LISTING_ADDRESS":"-","COMPANY_ADDRESS":"上海市长宁路1033号联通大厦25楼","STATE_CODE_B_DESC":"-","CSRC_GREAT_CODE_DESC":

In [495]:
print(COMPANY_CODE,COMPANY_ABBR,FULLNAME)

['600050'] ['中国联通'] ['中国联合网络通信股份有限公司']


In [654]:
def get_url_company(url):
    return 'http://query.sse.com.cn/commonQuery.do?jsonCallBack=jsonpCallback00001&isPagination=false&sqlId=COMMON_SSE_ZQPZ_GP_GPLB_C&productid='+url+'&_=11111111111111'

def get_url_executive(url):
    return 'http://query.sse.com.cn/commonQuery.do?jsonCallBack=jsonpCallback80740&isPagination=true&productid='+url+'&sqlId=COMMON_SSE_ZQPZ_GG_GGRYLB_L&pageHelp.pageSize=25&pageHelp.pageCount=50&pageHelp.pageNo=1&pageHelp.beginPage=1&pageHelp.cacheSize=1&pageHelp.endPage=5&_=1535506282048'

def collect_sse():#调取30页，相当于主函数
    company_code=[]
    company_abbr=[]
    fullname=[]
    executives=[]
    business=[]
    start_time=[]
    index=1
    for x in company_codes:
        url_company=get_url_company(x)
        response_company=requests.get(url_company,headers=header_company)
        response_company=response_company.text[19:-1]
        json_company=json.loads(response_company)
        COMPANY_CODE = jsonpath(json_company,'$..result..COMPANY_CODE')
        COMPANY_ABBR = jsonpath(json_company,'$..result..COMPANY_ABBR')
        FULLNAME = jsonpath(json_company,'$..result..FULLNAME')
        
        url_executives=get_url_executive(x)
        response_executives=requests.get(url_executives,headers=header_company)
        response_executives=response_executives.text[19:-1]
        json_executives=json.loads(response_executives)
        EXECUTIVES = jsonpath(json_executives,'$..result..NAME')
        BUSINESS = jsonpath(json_executives,'$..result..BUSINESS')
        START_TIME = jsonpath(json_executives,'$..result..START_TIME')
        
        company_code.append(COMPANY_CODE)
        company_abbr.append(COMPANY_ABBR)
        fullname.append(FULLNAME)
        executives.append(EXECUTIVES)
        business.append(BUSINESS)
        start_time.append(START_TIME)
        print (index, COMPANY_CODE,COMPANY_ABBR,FULLNAME,EXECUTIVES,BUSINESS,START_TIME)
        index+=1
        
    return (company_code,company_abbr,fullname,executives,business,start_time)

In [655]:
start_time = time.time()
comp_code,comp_abbr,full_name,senior,occupation,data_since=collect_sse()
cost_time = time.time()-start_time
print(str(cost_time)+'s')

1 ['600000'] ['浦发银行'] ['上海浦东发展银行股份有限公司'] ['刘信义', '王新浩', '崔炳文', '刘以研', '潘卫东', '崔炳文', '谢伟', '王新浩', '徐海燕', '徐海燕'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副行长、党委委员', '副行长、党委委员'] ['2016-04-28', '2016-04-29', '2016-04-29', '2016-04-29', '2016-04-28', '2016-04-29', '2016-04-29', '2016-04-29', '2016-04-29', '2016-04-29']
2 ['600004'] ['白云机场'] ['广州白云国际机场股份有限公司'] ['谢冰心', '王晓勇', '黄浩', '武宇', '王晓勇', '于洪才', '戚耀明', '戚耀明', '戚耀明', '莫名贞'] ['总经济师', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '董事会秘书', '财务总监'] ['2016-12-07', '2016-08-30', '2016-12-07', '2016-12-07', '2016-08-30', '2016-12-07', '2016-12-07', '2016-12-07', '2016-12-07', '2016-12-07']
3 ['600006'] ['东风汽车'] ['东风汽车股份有限公司'] ['丁绍斌', '丁绍斌', '柯钢', '李祥平', '李争荣', '周晓伏', '张斌', '张斌'] ['总经理', '总经理', '总会计师', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2016-06-21', '2016-06-21', '2018-03-13', '2017-10-10', '2015-07-24', '2017-06-01', '2016-05-24', '2016-05-24']
4 ['600007'] ['中国国贸'] ['中国国际贸易中心股份有限公司'] ['唐炜', '林南春', '

28 ['600036'] ['招商银行'] ['招商银行股份有限公司'] ['施顺华', '汪建中', '施顺华', '田惠宇', '熊良俊', '连柏林', '李浩', '王良', '赵驹', '刘建军', '刘建军', '朱琦', '唐志宏', '李浩'] ['-', '-', '-', '总经理', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副行长兼永隆银行行政总裁', '副行长', '副行长'] ['2017-04-01', '2017-04-01', '2017-04-01', '2016-06-28', '2014-09-01', '2012-06-01', '2007-10-22', '2016-06-28', '2015-02-10', '2013-12-11', '2013-12-04', '2008-09-01', '2010-06-23', '2001-12-01']
29 ['600037'] ['歌华有线'] ['北京歌华有线电视网络股份有限公司'] ['卢东涛', '卢东涛', '胡志鹏', '曾春', '曾春', '梁彦军', '唐文伟', '康朝晖', '何拥军', '唐文伟'] ['总经理', '总经理', '总会计师', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2013-03-27', '2013-03-27', '2013-03-27', '2014-01-16', '2014-01-16', '2013-03-27', '2014-01-16', '2013-03-27', '2013-03-27', '2014-01-16']
30 ['600038'] ['中直股份'] ['中航直升机股份有限公司'] ['吕杰', '堵娟', '堵娟', '吴希明', '吴希明', '裴家利', '聂小铭', '洪蛟', '陈光', '夏群林', '孙庆民', '裴家利', '陈光', '孙庆民', '夏群林', '周新民', '陈光', '张继超', '顾韶辉'] ['总经理', '总会计师', '总会计师', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总

57 ['600075'] ['新疆天业'] ['新疆天业股份有限公司'] ['关刚', '张强', '李熙桉', '张强', '李熙桉', '严健', '李刚', '夏中兵'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28']
58 ['600076'] ['康欣新材'] ['康欣新材料股份有限公司'] ['杨宾', '杨其礼', '杨宾', '杨刚', '杨其礼', '王粟旸', '杨刚', '牟儆', '牟儆', '李武鹏', '李武鹏'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监', '财务总监'] ['2015-12-29', '2015-12-29', '2015-12-29', '2015-12-29', '2015-12-29', '2017-05-22', '2015-12-29', '2015-12-29', '2015-12-29', '2015-12-29', '2015-12-29']
59 ['600077'] ['宋都股份'] ['宋都基业投资股份有限公司'] ['俞建午', '汪庆华', '戴克强', '戴克强', '陈振宁', '蒋燚俊', '郑羲亮', '郑羲亮'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2016-04-08', '2016-04-08', '2016-04-08', '2016-04-08', '2016-04-08', '2016-04-08', '2016-04-08', '2016-04-08']
60 ['600078'] ['澄星股份'] ['江苏澄星磷化工股份有限公司'] ['傅本度', '傅本度', '赵俊丰', '赵俊丰', '钱文贤', '夏正华', '夏正华', '周忠明'] [

90 ['600111'] ['北方稀土'] ['中国北方稀土(集团)高科技股份有限公司'] ['李金玲', '刘义', '刘义', '王臣', '王臣', '余英武', '王占成', '王占成'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2018-02-28', '2014-12-29', '2014-12-29', '2016-09-06', '2016-09-06', '2018-05-29', '2016-09-06', '2016-09-06']
91 ['600112'] ['天成控股'] ['贵州长征天成控股股份有限公司'] ['曹玉生'] ['副总经理'] ['2017-12-11']
92 ['600113'] ['浙江东日'] ['浙江东日股份有限公司'] ['戴元仁', '庄克进', '戴元仁', '赵阿宝', '庄克进', '谢小磊', '谢小磊'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2016-05-13', '2016-05-13', '2016-05-13', '2018-02-09', '2016-05-13', '2016-05-13', '2016-05-13']
93 ['600114'] ['东睦股份'] ['东睦新材料集团股份有限公司'] ['朱志荣', '许凯', '曹阳', '多田昌弘', '何灵敏', '多田昌弘', '肖亚军'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2016-08-06', '2016-08-06', '2016-08-06', '2016-08-06', '2016-08-06', '2016-08-06', '2016-08-06']
94 ['600115'] ['东方航空'] ['中国东方航空股份有限公司'] ['吴永良', '姜疆', '冯亮', '冯德华', '姜疆', '马须伦', '汪健', '唐兵', '李养民'] ['副总经理、财务总监', '副总经理', '副总经理', '副总经理', '副总经理', '副董事长、总经理'

121 ['600148'] ['长春一东'] ['长春一东离合器股份有限公司'] ['孟庆洪', '孟庆洪', '李文涛', '孙长增', '郭杰', '郭杰', '白光申'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2016-02-25', '2016-02-25', '2014-04-24', '2017-04-21', '2012-04-13', '2012-04-13', '2012-08-16']
122 ['600149'] ['ST坊展'] ['廊坊发展股份有限公司'] ['魏树焕', '曹玫', '张春岭', '张春岭'] ['其他', '副董事长', '董事会秘书', '董事会秘书'] ['2016-08-19', '2013-08-19', '2016-08-19', '2016-08-19']
123 ['600150'] ['*ST船舶'] ['中国船舶工业股份有限公司'] ['陈琼', '陈琼', '陶健', '陶健'] ['总会计师', '总会计师', '董事会秘书', '董事会秘书'] ['2015-03-03', '2015-03-03', '2015-03-03', '2015-03-03']
124 ['600151'] ['航天机电'] ['上海航天汽车机电股份有限公司'] ['张建功', '贺宁坡', '李亚军', '金琪'] ['总经理', '总会计师', '副总经理', '副总经理'] ['2017-03-07', '2018-03-23', '2018-03-23', '2015-06-10']
125 ['600152'] ['维科技术'] ['维科技术股份有限公司'] ['黄福良', '薛春林'] ['副总经理', '财务总监'] ['2017-10-16', '2017-10-16']
126 ['600153'] ['建发股份'] ['厦门建发股份有限公司'] ['郑永达', '王志兵', '高莉娟', '王志兵', '陈东旭', '赖衍达', '高莉娟', '林茂', '江桂芝', '许加纳', '许加纳'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', 

156 ['600187'] ['国中水务'] ['黑龙江国中水务股份有限公司'] ['王霞', '庄建龙', '丁宏伟', '周吉全', '丁宏伟'] ['-', '副总经理', '副总经理', '副总经理', '副总经理'] ['2018-04-19', '2018-04-19', '2018-04-19', '2018-04-19', '2018-04-19']
157 ['600188'] ['兖州煤业'] ['兖州煤业股份有限公司'] ['吴向前', '王富奇', '赵洪刚', '刘健', '贺敬', '刘健', '靳庆彬', '赵青春', '赵青春'] ['总经理', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29']
158 ['600189'] ['吉林森工'] ['吉林森林工业股份有限公司'] ['王海', '刘波', '王炳平', '王尽晖', '王尽晖', '包卓', '时军', '薛义'] ['-', '-', '-', '-', '-', '总经理', '董事会秘书', '财务总监'] ['2016-09-26', '2016-09-26', '2016-09-26', '2016-09-26', '2016-09-26', '2016-09-26', '2016-09-26', '2016-09-26']
159 ['600190'] ['锦州港'] ['锦州港股份有限公司'] ['刘辉', '刘辉', '张文博', '张文博', '王兴山', '李志超', '王鸿', '詹炜', '刘福金', '王鸿', '王兴山', '刘亚忠', '宁鸿鹏', '李桂萍', '李挺', '李挺'] ['总经理', '总经理', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '

189 ['600225'] ['天津松江'] ['天津松江股份有限公司'] ['刘新林', '刘大庆', '詹鹏飞', '殷尚宏', '刘大庆', '詹鹏飞', '苏桓', '苏桓'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2017-04-06', '2016-08-24', '2016-08-24', '2016-08-24', '2016-08-24', '2016-08-24', '2016-08-24', '2016-08-24']
190 ['600226'] ['瀚叶股份'] ['浙江瀚叶股份有限公司'] ['孙文秋', '陶舜晓', '翁天雄', '陶舜晓', '翁天雄', '李晓伟', '王旭光', '吴昶'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2017-05-15', '2017-05-15', '2016-04-08', '2017-05-15', '2016-04-08', '2017-05-15', '2017-05-15', '2017-05-15']
191 ['600227'] ['圣济堂'] ['贵州赤天化股份有限公司'] ['高敏红', '高敏红', '袁远镇', '胡正学', '王蓉'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2015-01-12', '2015-01-12', '2008-04-22', '2017-08-04', '2017-08-04']
192 ['600228'] ['ST昌九'] ['江西昌九生物化工股份有限公司'] ['卢岐', '李季'] ['总经理', '董事会秘书'] ['2018-04-26', '2018-01-10']
193 ['600229'] ['城市传媒'] ['青岛城市传媒股份有限公司'] ['杨延亮', '李楷', '张化新', '马琪', '苏彩霞', '苏彩霞'] ['总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2015-09-02', '2015-09-02', '2015-09

221 ['600261'] ['阳光照明'] ['浙江阳光照明电器集团股份有限公司'] ['官勇', '陈以平', '许文平', '李阳', '吴国明', '赵芳华', '周亚梅', '周亚梅'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2017-05-09', '2017-05-09', '2017-05-09', '2017-05-09', '2017-05-09', '2017-05-09', '2017-05-09', '2017-05-09']
222 ['600262'] ['北方股份'] ['内蒙古北方重型汽车股份有限公司'] ['邬青峰', '邬青峰', '张海楠', '郭海全', '巴根', '薛继奎', '常德明', '常德明', '侯文瑞'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2016-05-20', '2016-05-20', '2018-03-27', '2018-03-27', '2016-05-20', '2016-05-20', '2016-05-20', '2016-05-20', '2016-05-20']
223 ['600265'] ['ST景谷'] ['云南景谷林业股份有限公司'] ['段贵祥', '段贵祥', '段攀', '段攀', '季苏亚', '季苏亚'] ['副总经理', '副总经理', '副总经理', '副总经理', '财务总监', '财务总监'] ['2017-06-15', '2017-06-15', '2017-06-15', '2017-06-15', '2017-06-15', '2017-06-15']
224 ['600266'] ['北京城建'] ['北京城建投资发展股份有限公司'] ['张财广', '储昭武', '梁伟明', '范亚斌', '蔡志强', '武涛', '李洪兵', '蔡志强', '武涛', '李洪兵', '李学富', '邹哲', '毛雅清', '王恒清', '王恒清', '程丰', '毛雅清', '肖红卫'] ['其他', '其他', '其他', '其他', '副总经理', 

253 ['600299'] ['安迪苏'] ['蓝星安迪苏股份有限公司'] ['Jean Marc Dublanc', '顾登杰', '梁齐南', '蔡昀'] ['总经理', '副总经理', '董事会秘书', '财务总监'] ['2015-10-23', '2015-10-23', '2018-03-20', '2015-10-23']
254 ['600300'] ['维维股份'] ['维维食品饮料股份有限公司'] ['赵惠卿', '张明扬', '藤谷阳一', '曹荣开', '孙欣', '孟召永', '赵昌磊', '赵昌磊'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2017-08-01', '2017-08-01', '2017-08-01', '2017-08-01', '2017-08-01', '2017-08-01', '2017-08-01', '2017-08-01']
255 ['600301'] ['ST南化'] ['南宁化工股份有限公司'] ['陈军', '陈军', '蔡桂生', '蔡桂生', '张海坚', '张海坚'] ['总工程师', '总工程师', '其他', '董事会秘书', '财务总监', '财务总监'] ['2016-12-15', '2016-12-15', '2018-06-21', '2016-12-15', '2016-12-15', '2016-12-15']
256 ['600302'] ['标准股份'] ['西安标准工业股份有限公司'] ['赵旭', '赵旭', '黄玮', '石磊', '原增胜', '石磊', '邓斌', '余守旗', '陈锦山', '郑璇'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监、董事会秘书'] ['2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23']
257 ['600

285 ['600335'] ['国机汽车'] ['国机汽车股份有限公司'] ['夏闻迪', '贾屹', '尹建弘', '田亚梅', '贾屹', '许全有', '田亚梅', '谈正国', '谈正国', '鲁德恒'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2015-12-07', '2015-12-23', '2015-12-23', '2015-12-23', '2015-12-23', '2015-12-23', '2015-12-23', '2015-12-07', '2015-12-07', '2015-12-23']
286 ['600336'] ['澳柯玛'] ['澳柯玛股份有限公司'] ['张斌', '徐玉翠', '刘金彬', '王英峰', '于正奇', 'KIM SANGUK(金湘彧)', 'KIM SANGUK(金湘彧)', '郑培伟', '李蔚'] ['总经理', '总会计师、财务负责人', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事长'] ['2017-05-19', '2017-05-19', '2017-09-01', '2017-05-19', '2017-09-01', '2017-05-19', '2017-05-19', '2017-09-01', '2006-04-30']
287 ['600337'] ['美克家居'] ['美克国际家居用品股份有限公司'] ['陈江', '顾少军', '赵晶', '黄新', '黄新', '张莉'] ['总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2017-12-07', '2017-12-07', '2017-12-07', '2017-12-07', '2017-12-07', '2017-12-07']
288 ['600338'] ['西藏珠峰'] ['西藏珠峰资源股份有限公司'] ['王喜兵', '王喜兵', '黄亚婷', '梁明', '孟宪党', '孟宪党', '任小华', '任小华', '胡晗东', '胡晗东', '张树祥', '张树祥'] [

319 ['600377'] ['宁沪高速'] ['江苏宁沪高速公路股份有限公司'] ['顾德军', '周崇明', '顾德军', '尚红', '李捷', '姚永嘉', '任卓华', '任卓华'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2016-10-27', '2018-03-23', '2016-10-27', '2007-11-01', '2012-08-17', '1997-06-27', '2018-06-04', '2018-06-04']
320 ['600378'] ['天科股份'] ['四川天一科技股份有限公司'] ['李守荣', '李守荣', '汤洪', '汤洪', '陈健', '郜豫川', '冯新华', '冯新华', '龙崇军', '龙崇军'] ['总经理', '总经理', '总工程师', '总工程师', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监', '财务总监'] ['2017-04-06', '2017-04-06', '2015-12-11', '2015-12-11', '2015-12-11', '2015-12-11', '2016-02-22', '2016-02-22', '2015-11-09', '2015-11-09']
321 ['600379'] ['宝光股份'] ['陕西宝光真空电器股份有限公司'] ['胡唯', '王荃', '王荃', '张军', '王庆成', '袁军为', '原瑞涛', '杜轶名'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2017-05-19', '2017-05-19', '2017-05-19', '2017-05-19', '2017-08-19', '2017-05-19', '2017-05-19', '2017-05-19']
322 ['600380'] ['健康元'] ['健康元药业集团股份有限公司'] ['赵凤光', '俞雄'] ['其他', '副总经理'] ['2016-10-26', '2016-10-26']
323 ['600381'] ['青海春

349 ['600416'] ['湘电股份'] ['湘潭电机股份有限公司'] ['杨达', '梁小波', '李怡文', '熊斌'] ['副总经理', '副总经理', '董事会秘书', '财务总监'] ['2018-04-23', '2018-04-23', '2018-04-23', '2018-04-23']
350 ['600418'] ['江淮汽车'] ['安徽江淮汽车集团股份有限公司'] ['李明', '陈志平', '冯梁森', '冯梁森', '陶伟'] ['副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2015-11-20', '2018-07-20', '2015-07-16', '2018-07-20', '2018-07-20']
351 ['600419'] ['天润乳业'] ['新疆天润乳业股份有限公司'] ['胡刚', '胡刚', '刘朋龙', '刘朋龙', '陈东升', '王慧玲', '王慧玲', '周建国', '尤宏', '尤宏', '陈东升', '王军', '周建国', '冯育菠'] ['总经理', '总经理', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2018-01-25', '2017-01-25', '2018-07-17']
352 ['600420'] ['现代制药'] ['上海现代制药股份有限公司'] ['杨军', '魏冬松', '魏冬松', '王述东', '王述东', '龚忠', '李显林', '崔昳昤', '邓宝军', '龚忠', '李显林', '崔昳昤', '邓宝军', '李昊', '李昊'] ['总经理', '副总经理、董事会秘书', '副总经理、董事会秘书', '副总经理', '副总经理', '副总经理', '副总经理', '副

381 ['600469'] ['风神股份'] ['风神轮胎股份有限公司'] ['申玉生', 'Paolo Benea', 'Paolo Ghirga', 'Paolo Benea', '张晓新', '刘新军', '袁亮'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2018-04-14', '2018-04-14', '2018-04-14', '2018-04-14', '2018-04-14', '2018-04-14', '2018-04-14']
382 ['600470'] ['六国化工'] ['安徽六国化工股份有限公司'] ['缪振虎', '徐东奎', '马健', '沈浩', '徐东奎', '胡珂', '许进冲', '张福华', '邢金俄', '秦红'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2016-05-06', '2013-12-13', '2014-07-25', '2013-01-18', '2013-12-13', '2017-03-04', '2017-03-04', '2017-03-04', '2013-01-18', '2013-01-18']
383 ['600475'] ['华光股份'] ['无锡华光锅炉股份有限公司'] ['钟文俊', '钟文俊', '孟雷金', '李雄伟', '曹剑', '周建伟'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监'] ['2016-05-12', '2016-05-12', '2017-08-24', '2017-08-24', '2018-07-20', '2016-05-12']
384 ['600476'] ['湘邮科技'] ['湖南湘邮科技股份有限公司'] ['张华', '张华', '付振翔', '付振翔', '孟京京', '刘朝晖', '刘朝晖'] ['总经理', '总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2016-03-28', '2016-03-28', '2016-03-

414 ['600510'] ['黑牡丹'] ['黑牡丹（集团）股份有限公司'] ['邓建军', '史荣飞', '周明', '赵文骏', '恽伶俐', '葛维龙', '葛维龙', '何晓晴', '陈强', '陈强'] ['其他', '副总经理', '副总经理', '副总经理', '副总经理', '副董事长', '副董事长', '董事会秘书', '财务总监', '财务总监'] ['2018-05-07', '2018-05-07', '2018-05-07', '2018-05-07', '2018-05-07', '2018-05-07', '2018-05-07', '2018-05-07', '2018-05-07', '2018-05-07']
415 ['600511'] ['国药股份'] ['国药集团药业股份有限公司'] ['李辉', '邵伟', '吴杰', '肖卓远', '唐磊', '叶彤', '吴杰', '沈黎新'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监'] ['2018-07-11', '2017-08-22', '2017-08-22', '2018-04-19', '2018-04-19', '2017-08-22', '2017-08-22', '2017-08-22']
416 ['600512'] ['腾达建设'] ['腾达建设集团股份有限公司'] ['杨九如', '陈德锋', '辛晓东', '孙九春', '孙九春', '叶丽君', '董敏', '蒋冬湖', '严炜雷', '董敏', '王士金'] ['总经理', '总会计师', '审计总监', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2016-11-18', '2016-11-18', '2016-11-18', '2016-11-18', '2016-11-18', '2016-11-18', '2016-11-18', '2016-11-18', '2016-11-18', '2016-11-18', '2016-11-18']
417 ['600513'] ['联环药业'] ['江苏联环药业股份有限公司'] ['吴

445 ['600547'] ['山东黄金'] ['山东黄金矿业股份有限公司'] ['王德煜', '王德煜', '王培月', '李涛', '宋增春', '宋增春', '李涛', '何吉平'] ['-', '-', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2013-10-15', '2013-10-15', '2016-05-16', '2013-10-15', '2016-05-16', '2016-05-16', '2013-10-15', '2013-10-15']
446 ['600548'] ['深高速'] ['深圳高速公路股份有限公司'] ['温珀玮', '孙策', '黄毕南', '罗琨', '罗琨'] ['副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2015-09-01', '2015-09-01', '2015-09-01', '2016-01-29', '2016-01-29']
447 ['600549'] ['厦门钨业'] ['厦门钨业股份有限公司'] ['钟可祥', '洪超额', '郭天煌', '洪超额', '许火耀'] ['副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2018-04-24', '2018-04-24', '2018-04-24', '2018-04-24', '2018-04-24']
448 ['600550'] ['保变电气'] ['保定天威保变电气股份有限公司'] ['刘淑娟', '何光盛', '刘东升', '张喜乐', '何光盛'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2018-02-09', '2018-02-09', '2018-02-09', '2018-02-09', '2018-02-09']
449 ['600551'] ['时代出版'] ['时代出版传媒股份有限公司'] ['张克文', '张克文', '刘红', '韩进', '包云鸠', '林清发', '桂宾'] ['其他', '其他', '副总经理、董事会秘书', '副总经理', '副总经理', '董事、总编辑', '财务总监'] ['2015-11-28', '201

476 ['600582'] ['天地科技'] ['天地科技股份有限公司'] ['肖宝贵', '黄乐亭', '李玉魁', '黄乐亭', '闫少宏', '范建', '宋家兴'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2014-06-27', '2014-06-27', '2016-04-27', '2014-06-27', '2014-06-27', '2014-06-27', '2014-06-27']
477 ['600583'] ['海油工程'] ['海洋石油工程股份有限公司'] ['金晓剑', '金晓剑', '陈荣旗', '陈荣旗', '陈宝洁', '于长生', '林瑶生', '陈宝洁', '李小巍', '李小巍', '林瑶生', '于长生', '刘连举', '陈永红'] ['总经理', '总经理', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2017-01-22', '2017-01-22', '2016-04-22', '2016-04-22', '2016-07-06', '2016-07-06', '2016-07-06', '2016-07-06', '2016-07-06', '2016-07-06', '2016-07-06', '2016-07-06', '2001-02-01', '2012-08-29']
478 ['600584'] ['长电科技'] ['江苏长电科技股份有限公司'] ['罗宏伟', '赖志明', '马思立', '田镇英', '徐良辉', '田镇英', '徐良辉', 'Shim Il Kwon', '王新潮', '朱正义'] ['其他', '其他', '其他', '其他', '其他', '其他', '其他', '其他', '其他', '董事会秘书'] ['2016-04-21', '2016-04-21', '2016-04-21', '2016-04-21', '2016-04-21', '2016-04-21', '2016-04-21', '2017-09-28', '2016-04-21',

512 ['600620'] ['天宸股份'] ['上海市天宸股份有限公司'] ['代峥嵘', '邬显顺', '邬显顺', '潘露', '潘露'] ['总会计师', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2017-03-07', '2017-03-07', '2017-03-07', '2017-03-07', '2017-03-07']
513 ['600621'] ['华鑫股份'] ['上海华鑫股份有限公司'] ['田明', '赵恒', '胡之奎', '胡之奎'] ['总会计师', '副总经理', '董事会秘书', '董事会秘书'] ['2017-05-15', '2017-05-15', '2017-05-15', '2017-05-15']
514 ['600622'] ['光大嘉宝'] ['光大嘉宝股份有限公司'] ['曹萍', '金红', '胡兵', '余小玲', '沈培新', '石建良', '沈培新', '石建良', '金红', '王幸千', '陈正友', '孙红良'] ['其他', '其他', '其他', '其他', '其他', '其他', '其他', '其他', '其他', '副总裁', '副总裁', '董事会秘书'] ['2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12', '2017-05-12']
515 ['600623'] ['华谊集团'] ['上海华谊集团股份有限公司'] ['王霞', '王霞', '魏建华', '顾立立', '陈耀', '黄德亨', '魏建华', '黄德亨', '顾春林', '方广清', '顾春林', '常达光', '常达光'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '董事会秘书', '财务总监', '财务总监'] ['2016-03-18', '2016-03-18', '2016-03-18', '2016-11

546 ['600661'] ['新南洋'] ['上海新南洋股份有限公司'] ['吴竹平', '刘江萍', '朱凯泳', '宋培林', '林涛', '杨夏'] ['总经理', '总会计师', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2017-09-25', '2017-09-25', '2017-09-25', '2017-09-25', '2017-09-25', '2017-09-25']
547 ['600662'] ['强生控股'] ['上海强生控股股份有限公司'] ['陈放', '杜慧', '程林', '杜慧', '陆立平', '陆立平', '刘红威', '刘纯'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2016-04-08', '2016-04-08', '2017-04-27', '2016-04-08', '2017-09-18', '2017-09-18', '2015-06-01', '2018-01-05']
548 ['600663'] ['陆家嘴'] ['上海陆家嘴金融贸易区开发股份有限公司'] ['周伟民', '周翔', '丁晓奋', '丁晓奋', '贾伟', '王辉', '胡习'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书、办公室主任', '财务总监'] ['2014-04-22', '2014-04-22', '2015-07-29', '2015-07-29', '2018-06-26', '2013-07-26', '2018-06-26']
549 ['600664'] ['哈药股份'] ['哈药集团股份有限公司'] ['张镇平', '吴志军', '刘波', '周行', '魏双莹', '沙梅', '孟晓东', '沙梅'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2017-10-25', '2017-10-25', '2017-10-25', '2017-10-25', '2017-10-25', '2017-10-25', '2017-10-25', '2017

582 ['600701'] ['*ST工新'] ['哈尔滨工大高新技术产业开发股份有限公司'] ['姚永发', '姚永发', '吕莹', '任会云', '王梅', '王梅'] ['总经理', '总经理', '副总经理兼董事会秘书', '副总经理', '财务总监', '财务总监'] ['2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29', '2017-06-29']
583 ['600702'] ['舍得酒业'] ['舍得酒业股份有限公司'] ['李强', '张树平', '郭建亚', '曾家斌', '蒲吉洲', '曾家斌', '李富全', '张树平', '郭建亚'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2017-05-26', '2017-05-26', '2017-05-26', '2017-05-26', '2017-05-26', '2017-05-26', '2017-05-26', '2017-05-26', '2017-05-26']
584 ['600703'] ['三安光电'] ['三安光电股份有限公司'] ['林科闯', '张家宏', '徐宸科', '苏住裕', '蔡文必', '张家宏', '徐宸科', '苏住裕', '蔡文必', '林海', '韦大曼', '林志东', '张中英', '张中英', '李雪炭', '李雪炭', '黄智俊'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2017-07-10', '2014-07-11', '2014-07-11', '2014-07-11', '2014-07-11', '2014-07-11', '2014-07-11', '2014-07-11', '2014-07-11', '2011-01-20', '2011-12-01', '2017-07-10', '2017-0

612 ['600732'] ['ST新梅'] ['上海新梅置业股份有限公司'] ['戚晓蓉', '李煜坤', '荀九斤', '荀九斤'] ['副总经理', '董事会秘书', '财务总监', '财务总监'] ['2018-05-25', '2016-03-17', '2016-11-17', '2016-11-17']
613 ['600733'] ['SST前锋'] ['北京前锋电子股份有限公司'] ['王俊亚', '王允慧'] ['-', '董事会秘书'] ['2018-04-19', '2018-01-16']
614 ['600734'] ['实达集团'] ['福建实达集团股份有限公司'] ['叶成辉', '汪清', '郭华', '汪清', '王立', '陈蓓', '吴波'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2017-11-28', '2017-04-10', '2018-02-13', '2017-04-10', '2017-04-10', '2018-02-13', '2017-04-10']
615 ['600735'] ['新华锦'] ['山东新华锦国际股份有限公司'] ['盛强', '禹晶', '禹晶'] ['董事会秘书', '财务总监', '财务总监'] ['2017-08-28', '2016-07-04', '2016-07-04']
616 ['600736'] ['苏州高新'] ['苏州新区高新技术产业股份有限公司'] ['王星', '贺明', '张晓峰', '贺明', '屈晓云', '徐征', '楼铭育', '宋才俊', '茅宜群'] ['总经理', '其他', '其他', '其他', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2016-07-22', '2016-05-13', '2017-04-05', '2016-05-13', '2017-06-09', '2017-06-09', '2015-09-30', '2015-09-30', '2015-09-30']
617 ['600737'] ['中粮糖业'] ['中粮屯河糖业股份有限公司'] ['赵玮', '于作江', '吴震', '于作江', '

648 ['600770'] ['综艺股份'] ['江苏综艺股份有限公司'] ['昝圣达', '陆建峰', '黄天鸿', '顾政巍', '陆建峰', '邢光新', '顾政巍', '钱志华'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务负责人'] ['2016-06-29', '2016-06-29', '2016-06-29', '2016-06-29', '2016-06-29', '2016-06-29', '2016-06-29', '2016-06-29']
649 ['600771'] ['广誉远'] ['广誉远中药股份有限公司'] ['柳花兰', '张懿', '王佩义'] ['-', '-', '副总经理'] ['2017-04-21', '2017-04-21', '2017-04-21']
650 ['600773'] ['西藏城投'] ['西藏城市发展投资股份有限公司'] ['王柏东', '程晓林', '于隽隽', '符蓉', '廖婷'] ['副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2018-05-11', '2018-05-11', '2018-05-11', '2018-05-11', '2018-05-11']
651 ['600774'] ['汉商集团'] ['武汉市汉商集团股份有限公司'] ['刘传致', '李邹强', '肖萍', '张竖珍', '张燕萍', '冯振宇'] ['总会计师', '其他', '其他', '其他', '其他', '副总经理'] ['2016-05-20', '2017-04-21', '2017-12-19', '2017-12-19', '2017-12-19', '2016-05-20']
652 ['600775'] ['南京熊猫'] ['南京熊猫电子股份有限公司'] ['夏德伟', '刘坤', '胡回春', '邵波', '邵波', '刘坤', '胡回春', '郭庆', '沈见龙'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2018-06-29', '2018-06-29', '

681 ['600808'] ['马钢股份'] ['马鞍山钢铁股份有限公司'] ['钱海帆', '田俊', '陆克从', '张文洋', '伏明', '张文洋', '田俊', '何红云'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2017-11-30', '2017-11-30', '2017-11-30', '2017-11-30', '2017-11-30', '2017-11-30', '2017-11-30', '2018-04-19']
682 ['600809'] ['山西汾酒'] ['山西杏花村汾酒厂股份有限公司'] ['常建伟', '马世彪', '郝光岭', '杜小威', '武爱东', '高志峰', '武爱东', '郝光岭', '高志峰', '武世杰', '李俊', '赵玲', '宋青年', '王涛'] ['总经理', '总会计师', '其他', '其他', '其他', '其他', '其他', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2016-03-01', '2005-07-01', '2016-05-16', '2005-07-01', '2016-05-16', '2016-05-16', '2016-05-16', '2016-05-16', '2016-05-16', '2012-09-09', '2018-01-22', '2012-09-09', '2018-05-11', '2015-03-12']
683 ['600810'] ['神马股份'] ['神马实业股份有限公司'] ['段文亮', '齐文兵', '张健', '张健', '齐文兵', '张鲁亚', '刘臻', '郑晓广', '赵运通', '赵运通'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事长', '财务总监', '财务总监'] ['2003-11-27', '2015-07-29', '2009-12-15', '2009-12-15', '2015-07-29', '2016-05-23', '2012-12-25', '2007-0

714 ['600845'] ['宝信软件'] ['上海宝信软件股份有限公司'] ['朱湘凯', '朱湘凯', '陈健', '宋健海', '王剑虎', '周建平', '胡国奋', '吕子男', '吕子男'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监', '财务总监'] ['2016-07-28', '2016-07-28', '2016-04-26', '2016-04-26', '2016-04-26', '2016-04-26', '2016-07-28', '2016-04-26', '2016-04-26']
715 ['600846'] ['同济科技'] ['上海同济科技实业股份有限公司'] ['张晔', '俞卫中', '杨卫东', '杨卫东', '骆君君', '骆君君'] ['总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2016-02-26', '2016-11-11', '2016-02-26', '2016-02-26', '2016-02-26', '2016-02-26']
716 ['600847'] ['万里股份'] ['重庆万里新能源股份有限公司'] ['张文江', '王永东', '张冰冰', '王永东', '李建科', '关兰英', '杜正洪'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '财务总监', '财务总监'] ['2018-08-17', '2016-07-07', '2016-07-07', '2018-08-17', '2015-11-26', '2018-08-17', '2015-11-26']
717 ['600848'] ['上海临港'] ['上海临港控股股份有限公司'] ['丁桂康', '朱俊', '刘德宏', '孙仓龙', '孙昂', '许兴虎', '张勇', '王春辉', '陆雯', '邓睿宗', '邓睿宗'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2016-10-27', '2017-05-25',

746 ['600880'] ['博瑞传播'] ['成都博瑞传播股份有限公司'] ['李翔宇', '徐晓东', 'l刘乃贵', '苟军'] ['副总经理', '副总经理', '副总经理', '董事会秘书'] ['2015-04-08', '2015-04-08', '2015-11-19', '2015-04-08']
747 ['600881'] ['亚泰集团'] ['吉林亚泰(集团)股份有限公司'] ['彭雪松', '宋尚龙', '李斌', '刘树森', '田奎武', '韩冬阳', '王化民', '刘晓峰', '张凤瑛', '陈继忠', '刘晓峰', '杨毅鸣', '秦音'] ['总会计师', '总裁', '其他', '副总经理', '副总经理', '副总经理', '副总裁、总经济师', '副总裁', '副总裁', '副总裁', '副总裁', '副总裁', '董事会秘书'] ['2017-12-22', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27', '2017-06-27']
748 ['600882'] ['广泽股份'] ['上海广泽食品科技股份有限公司'] ['柴琇', '任松', '胡彦超', '郭永来', '刘宗尚', '邹有伟', '郭永来', '胡彦超', '刘宗尚', '胡彦超', '白丽君'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监'] ['2015-10-08', '2016-01-18', '2017-01-18', '2017-01-18', '2017-01-18', '2018-07-03', '2017-01-18', '2017-01-18', '2017-01-18', '2017-01-18', '2015-10-08']
749 ['600883'] ['博闻科技'] ['云南博闻科技实业股份有限公司'] ['施阳', '王春城', '杨庆

777 ['600958'] ['东方证券'] ['东方证券股份有限公司'] ['杨斌', '杨玉成', '舒宏', '舒宏', '杨玉成', '王如富', '王如富', '张建辉', '张建辉'] ['其他', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监', '财务总监'] ['2018-03-29', '2018-03-29', '2018-03-29', '2018-03-29', '2018-03-29', '2018-03-29', '2018-03-29', '2018-03-29', '2018-03-29']
778 ['600959'] ['江苏有线'] ['江苏省广电有线信息网络股份有限公司'] ['许如钢', '钱进', '陈健', '陈健', '钱进', '许如钢', '陈侃晔', '陈侃晔'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2008-04-01', '2008-04-01', '2013-05-01', '2013-05-01', '2008-04-01', '2008-04-01', '2016-06-24', '2016-06-24']
779 ['600960'] ['渤海汽车'] ['渤海汽车系统股份有限公司'] ['林风华', '王云刚', '朱昱', '季军', '王洪波'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2018-05-21', '2018-05-21', '2018-05-21', '2018-05-21', '2018-05-21']
780 ['600961'] ['株冶集团'] ['株洲冶炼集团股份有限公司'] ['刘朗明', '李雄姿', '刘文德', '刘文德', '何献忠', '何献忠'] ['总经理', '副总经理', '副总经理', '副总经理', '财务总监', '财务总监'] ['2017-03-09', '2013-11-08', '2007-10-12', '2007-10-12', '2010-10-25', '2010-10-25']
781 ['600962'] ['国

808 ['600996'] ['贵广网络'] ['贵州省广播电视信息网络股份有限公司'] ['李巍', '旷宗仁', '魏奇志', '林世凯', '徐军', '黄宗文', '翟海虹'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2016-07-16', '2016-07-16', '2016-07-16', '2016-07-16', '2016-07-16', '2016-07-16', '2016-07-16']
809 ['600997'] ['开滦股份'] ['开滦能源化工股份有限公司'] ['曹玉忠', '董立满', '董立满', '王发良', '梁希峰', '王发良', '侯树忠', '王连灵', '侯占友', '房承宣', '王连灵', '孙汉玉', '梁希峰'] ['总经理', '总会计师', '总会计师', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28', '2017-12-28']
810 ['600998'] ['九州通'] ['九州通医药集团股份有限公司'] ['龚翼华', '陈启明', '刘登攀', '郭磊', '郭磊', '陈松柏', '刘登攀', '王启兵', '杨菊美', '杨菊美', '刘义常', '林新扬', '许应政'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2017-11-21', '2017-11-21', '2017-11-21', '2017-11-21', '2017-11-21', '2017-11-21', '2017-11-

837 ['601098'] ['中南传媒'] ['中南出版传媒集团股份有限公司'] ['丁双平', '刘清华', '刘清华', '陈昕', '黄楚芳', '高军', '王丽波'] ['总经理', '其他', '其他', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2008-12-25', '2008-12-25', '2008-12-25', '2015-08-24', '2015-08-24', '2008-12-25', '2008-12-25']
838 ['601099'] ['太平洋'] ['太平洋证券股份有限公司'] ['李长伟', '程绪兰', '康静', '康静', '许弟伟', '史明坤', '张洪斌', '张东海', '聂愿牛', '唐卫华', '周岚'] ['总经理', '其他', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2016-07-05', '2018-05-21', '2017-04-30', '2017-04-30', '2016-07-05', '2016-07-05', '2016-07-05', '2016-07-05', '2016-07-05', '2016-07-05', '2016-07-05']
839 ['601100'] ['恒立液压'] ['江苏恒立液压股份有限公司'] ['邱永宁', '邱永宁', '赵雪阳', '胡国享', '王斌', '胡国享', '徐进', '张小芳', '张小芳', '彭玫'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2016-09-07', '2016-09-07', '2016-09-07', '2016-09-07', '2016-09-07', '2016-09-07', '2016-09-07', '2018-02-28', '2018-02-28', '2017-10-30']
840 ['601101'] ['昊华能源'] ['北京昊华能源股份有限公司'] ['张伟', '马植胜', '娄英杰', '关杰', '相啸宇',

866 ['601199'] ['江南水务'] ['江苏江南水务股份有限公司'] ['华锋', '华锋', '吴耀东', '吴耀东', '曾武', '陆庆喜', '陆庆喜', '高立', '朱杰'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2016-09-23', '2016-09-23', '2015-08-24', '2015-08-24', '2015-08-24', '2016-09-23', '2016-09-23', '2015-08-24', '2015-08-24']
867 ['601200'] ['上海环境'] ['上海环境集团股份有限公司'] ['王德浩', '叶辉', '邹庐泉', '张春明', '彭小平'] ['-', '-', '-', '董事会秘书', '财务总监'] ['2017-02-28', '2017-02-28', '2017-02-28', '2017-02-28', '2017-02-28']
868 ['601208'] ['东材科技'] ['四川东材科技集团股份有限公司'] ['曹学', '曹学', '李刚', '宗跃强', '宗跃强', '陈杰', '陈杰'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2016-11-14', '2016-11-14', '2016-11-14', '2016-11-14', '2016-11-14', '2016-11-14', '2016-11-14']
869 ['601211'] ['国泰君安'] ['国泰君安证券股份有限公司'] ['王松', '刘桂芳', '蒋忆明', '陈煜涛', '龚德雄', '朱健', '龚德雄', '朱健', '喻健', '谢乐斌', '谢乐斌'] ['总经理', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2015-08-21', '2008-12-03', '2013-11-22', '2016-11-28', '2016-11-28', '201

893 ['601368'] ['绿城水务'] ['广西绿城水务股份有限公司'] ['贝德光', '严红兵', '许雪菁'] ['其他', '副总经理', '董事会秘书'] ['2018-07-19', '2018-07-19', '2018-07-19']
894 ['601369'] ['陕鼓动力'] ['西安陕鼓动力股份有限公司'] ['李付俊', '陈党民', '陈党民', '陈余平', '李付俊', '蔡新平', '刘海军', '杨凯', '李广友', '杨季初', '杨季初', '赵甲文'] ['-', '总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2018-06-11', '2004-04-21', '2018-06-11', '2018-06-11', '2016-06-03', '2018-06-11', '2018-06-11', '2018-06-11', '2018-06-11', '2016-03-16', '2018-06-11', '2018-06-11']
895 ['601375'] ['中原证券'] ['中原证券股份有限公司'] ['李昭欣', '王静', '王静', '花金钟', '朱建民', '朱军红', '徐海军', '谢雪竹', '赵丽峰', '谢雪竹', '徐海军', '朱军红', '朱建民', '朱启本', '朱启本'] ['总会计师', '其他', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2018-01-29', '2018-01-18', '2018-01-18', '2018-04-12', '2007-08-01', '2009-09-01', '2014-07-01', '2012-08-01', '2012-08-01', '2012-08-01', '2014-07-01', '2009-09-01', '2007-08-01', '2018-01-18', '2018-01-18']
896 ['60137

922 ['601633'] ['长城汽车'] ['长城汽车股份有限公司'] ['王凤英', '李凤珍', '赵国庆', '郑春来', '郑春来', '赵国庆', '郝建军', '张文辉', '郝建军', '胡树杰', '胡树杰', '徐辉'] ['总经理', '副总经理、财务总监', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11', '2017-05-11']
923 ['601636'] ['旗滨集团'] ['株洲旗滨集团股份有限公司'] ['张柏忠', '胡勇', '凌根略', '宋辉', '姚培武', '姚培武', '张国明'] ['总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2017-03-29', '2017-03-29', '2018-04-09', '2018-04-09', '2017-03-29', '2017-03-29', '2017-03-29']
924 ['601666'] ['平煤股份'] ['平顶山天安煤业股份有限公司'] ['陶建平', '陶建平', '程伟', '岳殿召', '吉如昇', '许尽峰'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2016-11-10', '2016-11-10', '2016-05-16', '2017-12-25', '2016-06-24', '2017-06-12']
925 ['601668'] ['中国建筑'] ['中国建筑股份有限公司'] ['王祥明', '王祥明', '李百安', '马泽平', '郑学选', '赵晓江', '黄克斯', '刘锦章', '郑学选', '赵晓江', '邵继江', '孟庆禹', '薛克庆', '薛克庆'] ['总

948 ['601838'] ['成都银行'] ['成都银行股份有限公司'] ['郑军', '魏小瑛', '罗结', '黄建军', '蔡兵', '周亚西', '李金明', '李婉容', '罗铮', '兰青'] ['其他', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25']
949 ['601857'] ['中国石油'] ['中国石油天然气股份有限公司'] ['蔺爱国', '田景惠', '田景惠', '侯启军', '孙龙德', '吴恩来', '柴守平', '柴守平'] ['总工程师', '副总经理', '副总经理', '副总经理', '副总裁', '董事会秘书', '财务总监', '财务总监'] ['2007-06-18', '2015-11-25', '2015-11-25', '2017-06-08', '2007-06-18', '2014-02-25', '2017-01-25', '2017-01-25']
950 ['601858'] ['中国科传'] ['中国科技出版传媒股份有限公司'] ['李锋', '闫向东', '陈亮', '张凡', '李锋', '彭斌', '彭斌', '张小凌', '胡华强', '张小凌', '胡华强', '石强', '石强'] ['-', '-', '-', '-', '-', '总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监', '财务总监'] ['2014-07-30', '2018-02-09', '2018-02-09', '2018-02-09', '2014-07-30', '2016-06-29', '2016-06-29', '2014-07-30', '2014-07-30', '2014-07-30', '2014-07-30', '2014-07-30', '2014-07-30']
951

975 ['601958'] ['金钼股份'] ['金堆城钼业股份有限公司'] ['杨国平', '杨国平', '余和明', '朱永安', '尹孝刚', '高小宁', '刘新录', '李哲', '尹孝刚', '高小宁', '李哲', '李辉', '李辉', '尹战劳'] ['总经理', '总经理', '总会计师', '总工程师', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2017-05-31', '2018-04-20']
976 ['601965'] ['中国汽研'] ['中国汽车工程研究院股份有限公司'] ['万鑫铭', '万鑫铭', '苏自力', '苏自力', '周舟', '周本学', '刘安民', '刘安民', '周舟', '谢飞', '李开国'] ['总经理', '总经理', '总会计师', '总会计师', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事长'] ['2016-12-28', '2016-12-28', '2016-12-28', '2016-12-28', '2016-12-28', '2016-12-28', '2016-12-28', '2016-12-28', '2016-12-28', '2016-12-28', '2016-12-28']
977 ['601966'] ['玲珑轮胎'] ['山东玲珑轮胎股份有限公司'] ['王锋', '王锋', '孙松涛', '刘占村', '孙松涛', '刘占村', '马越川', '洪盟'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2016-06-20', '2016-06-20', '2016-06

1004 ['603016'] ['新宏泰'] ['无锡新宏泰电器科技股份有限公司'] ['赵敏海', '赵敏海', '冯伟祖', '陈建平', '沈华', '陈建平', '冯伟祖', '沈华', '杜建平', '杜建平', '高岩敏'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2017-11-02', '2017-11-02', '2017-11-02', '2017-11-02', '2017-11-02', '2017-11-02', '2017-11-02', '2017-11-02', '2017-11-02', '2017-11-02', '2017-11-02']
1005 ['603017'] ['中衡设计'] ['中衡设计集团股份有限公司'] ['张谨', '詹新建', '张延成', '蒋文蓓', '李铮', '胡义新', '于吉鹏', '胡义新', '于吉鹏', '徐海英'] ['-', '-', '-', '-', '-', '-', '-', '-', '-', '-'] ['2017-09-28', '2017-09-28', '2017-09-28', '2017-09-28', '2017-09-28', '2017-09-28', '2017-09-28', '2017-09-28', '2017-09-28', '2017-09-28']
1006 ['603018'] ['中设集团'] ['中设设计集团股份有限公司'] ['刘鹏', '明图章', '侯力纲', '凌九忠', '范东涛', '姚宇', '范东涛', '徐一岗', '姚宇', '胡安兵', '徐一岗'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理'] ['2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '2017-01-25', '

1037 ['603063'] ['禾望电气'] ['深圳市禾望电气股份有限公司'] ['陈文锋', '肖安波', '王永', '蔡海涛', '韩玉', '夏泉波', '郑大鹏', '刘济洲'] ['-', '-', '-', '-', '总经理', '副总经理', '副总经理', '董事会秘书'] ['2018-03-30', '2018-03-30', '2018-03-30', '2018-03-30', '2018-03-30', '2018-03-30', '2018-03-30', '2018-03-30']
1038 ['603066'] ['音飞储存'] ['南京音飞储存设备（集团）股份有限公司'] ['徐秦烨'] ['董事会秘书'] ['2016-04-14']
1039 ['603067'] ['振华股份'] ['湖北振华化学股份有限公司'] ['石义朗', '杨帆', '杨帆'] ['副总经理', '财务总监', '财务总监'] ['2018-04-25', '2018-04-25', '2018-04-25']
1040 ['603069'] ['海汽集团'] ['海南海汽运输集团股份有限公司'] ['林顺雄', '牛干', '林顺雄', '符人恩'] ['副总经理', '副总经理', '副总经理', '董事会秘书'] ['2009-11-01', '2009-11-01', '2009-11-01', '2017-10-26']
1041 ['603076'] ['乐惠国际'] ['宁波乐惠国际工程装备股份有限公司'] ['黄粤宁', '刘飞', '万财飞', '于化和', '董向阳', '吴再红', '吴勃'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2015-11-25', '2015-11-25', '2018-08-10', '2015-11-25', '2018-04-25', '2018-04-25', '2017-02-26']
1042 ['603077'] ['和邦生物'] ['四川和邦生物科技股份有限公司'] ['杨惠容', '刘安平', '王军', '王亚西', '莫融'] ['总经理', '副总经理', '副总经理', '副总经理', '董

1069 ['603126'] ['中材节能'] ['中材节能股份有限公司'] ['董兰起', '董兰起', '葛立武', '魏文华', '杨泽学', '葛立武', '魏文华', '杨泽学', '焦二伟'] ['其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监'] ['2017-11-27', '2017-11-27', '2017-11-27', '2017-11-27', '2017-11-27', '2017-11-27', '2017-11-27', '2017-11-27', '2017-11-27']
1070 ['603127'] ['昭衍新药'] ['北京昭衍新药研究中心股份有限公司'] ['左从林', '姚大林', '孙云霞', '高大鹏'] ['总经理', '副总经理', '副总经理', '财务总监'] ['2015-12-09', '2015-12-09', '2015-12-09', '2015-12-09']
1071 ['603128'] ['华贸物流'] ['港中旅华贸国际物流股份有限公司'] ['蔡显忠', '蔡显忠', '刘庆武', '于永乾', '于永乾'] ['副总经理', '副总经理', '副总经理', '财务总监', '财务总监'] ['2016-10-20', '2016-10-20', '2016-10-20', '2016-10-20', '2016-10-20']
1072 ['603129'] ['春风动力'] ['浙江春风动力股份有限公司'] ['竺鸣', '袁章平', '马刚杰', '陈柯亮', '倪树祥', '陈志勇', '周雄秀'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2015-12-19', '2015-12-19', '2015-12-19', '2018-07-24', '2015-12-19', '2018-07-04', '2015-12-19']
1073 ['603131'] ['上海沪工'] ['上海沪工焊接集团股份有限公司'] ['舒振宇', '舒振宇', '余定辉', '余定辉', '曹陈', '曹陈', '陈珩'] ['总经

1104 ['603203'] ['快克股份'] ['快克智能装备份有限公司'] ['戚国强', '戚国强', '窦小明', '刘志宏', '窦小明', '刘志宏', '苗小鸣', '苗小鸣'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2012-12-29', '2012-12-29', '2012-12-29', '2012-12-29', '2012-12-29', '2012-12-29', '2013-04-28', '2013-04-28']
1105 ['603208'] ['江山欧派'] ['江山欧派门业股份有限公司'] ['王忠', '王忠', '吴水燕', '汪志明', '刘佶南', '刘佶南', '汪志明', '吴水燕', '郑宏有', '方玉意', '方玉意'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2017-08-25', '2017-08-25', '2017-08-25', '2017-08-25', '2017-08-25', '2017-08-25', '2017-08-25', '2017-08-25', '2017-08-25', '2017-08-25', '2017-08-25']
1106 ['603214'] ['爱婴室'] ['上海爱婴室商务服务股份有限公司'] ['王云', '高岷', '乐人军'] ['副总经理', '董事会秘书', '财务总监'] ['2017-10-27', '2017-10-27', '2017-10-27']
1107 ['603218'] ['日月股份'] ['日月重工股份有限公司'] ['范信龙', '傅明康', '王烨', '傅明康', '王烨', '虞洪康', '范信龙', '虞洪康'] ['-', '-', '-', '-', '-', '-', '-', '-'] ['2007-12-01', '2007-12-01', '2010-12-01', '2007-12-01', '2010-12-01', '2007-12-01', '2007-12-

1136 ['603305'] ['旭升股份'] ['宁波旭升汽车技术股份有限公司'] ['徐旭东', '徐曦东', '钱海明', '陈兴方', '林国峰', '周小芬', '卢建波'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2018-08-10', '2018-08-10', '2018-08-10', '2018-08-10', '2018-08-10', '2018-08-10', '2018-08-10']
1137 ['603306'] ['华懋科技'] ['华懋（厦门）新材料科技股份有限公司'] ['张初全', '崔广三', '曹耀峰', '陈少琳'] ['总经理', '副总经理', '副总经理', '财务总监'] ['2010-06-28', '2015-07-06', '2013-10-08', '2010-06-28']
1138 ['603308'] ['应流股份'] ['安徽应流机电股份有限公司'] ['杜应流', '杜应流', '徐卫东', '丁邦满', '丁邦满', '程玉江', '沈厚平', '沈厚平', '徐卫东', '程玉江', '林欣', '林欣', '钟为义', '钟为义'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监', '财务总监'] ['2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07', '2017-04-07']
1139 ['603309'] ['维力医疗'] ['广州维力医疗器械股份有限公司'] ['韩广源', '韩广源', '陈云桂', '段嵩枫', '朱晓光', '朱晓光', '陈云桂', '段嵩枫', '张增勇', '张增勇', '谭文晖', '谭文晖'] ['总经理',

1168 ['603360'] ['百傲化学'] ['大连百傲化学股份有限公司'] ['刘海龙', '顾振鹏', '薛春华', '杨杰', '杨杰', '薛春华', '顾振鹏', '李喆', '王希梁'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2018-07-02', '2015-06-29', '2015-06-29', '2015-06-29', '2018-07-02', '2018-07-02', '2018-07-02', '2018-07-02', '2018-07-02']
1169 ['603363'] ['傲农生物'] ['福建傲农生物科技集团股份有限公司'] False False False
1170 ['603365'] ['水星家纺'] ['上海水星家用纺织品股份有限公司'] ['李裕陆', '李裕高', '李来斌', '沈义贵', '周忠', '孙子刚'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监'] ['2010-06-18', '2010-06-18', '2017-06-01', '2010-06-18', '2010-06-18', '2010-06-18']
1171 ['603366'] ['日出东方'] ['日出东方太阳能股份有限公司'] ['万旭昶', '焦青太', '李骏', '陈荣华', '张亚明', '朱亚林', '刘文玲', '徐新建', '李立干', '李立干'] ['总经理', '副总经理、总工程师', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事长', '财务总监', '财务总监'] ['2016-09-19', '2016-09-19', '2013-09-11', '2016-09-19', '2016-09-19', '2017-12-26', '2017-12-26', '2010-08-21', '2016-09-19', '2016-09-19']
1172 ['603367'] ['辰欣药业'] ['辰欣药业股份有限公司'] ['崔效廷', '孙洪晖'] ['副总经理', '董事会秘书'] [

1200 ['603500'] ['祥和实业'] ['浙江天台祥和实业股份有限公司'] ['汤啸', '赖金广', '汤文鸣', '汤克满', '鲍晓华', '陈英', '郑远飞'] ['总经理', '其他', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2015-09-10', '2015-09-10', '2015-09-10', '2015-09-10', '2018-04-19', '2018-05-11', '2015-09-10']
1201 ['603501'] ['韦尔股份'] ['上海韦尔半导体股份有限公司'] ['纪刚', '贾渊'] ['副总经理', '董事会秘书'] ['2016-03-21', '2016-03-21']
1202 ['603505'] ['金石资源'] ['金石资源集团股份有限公司'] ['戴隆松', '徐春波', '王福良', '杨晋', '戴水君', '武灵一'] ['其他', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2013-06-01', '2016-12-10', '2018-05-30', '2016-12-10', '2016-05-17', '2018-05-30']
1203 ['603506'] ['南都物业'] ['南都物业服务股份有限公司'] ['沈慧芳', '徐静', '潘才平', '张林', '吁悦', '余剑义', '赵磊', '陈红'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2015-12-27', '2015-12-27', '2015-12-27', '2016-09-09', '2017-11-10', '2017-07-05', '2018-03-30', '2016-03-02']
1204 ['603507'] ['振江股份'] ['江苏振江新能源装备股份有限公司'] ['徐建华'] ['副总经理'] ['2017-12-06']
1205 ['603508'] ['思维列控'] ['河南思维自动化设备股份有限公司'] ['石战成', '徐景胜', '焦炳岩', '秦伟', '秦伟', '高亚举'

1233 ['603587'] ['地素时尚'] ['地素时尚股份有限公司'] ['江瀛', '彭正昌', '马丽敏', '马瑞敏'] ['其他', '其他', '副董事长', '董事长'] ['2016-05-07', '2016-05-07', '2016-05-07', '2016-05-07']
1234 ['603588'] ['高能环境'] ['北京高能时代环境技术股份有限公司'] ['陈望明', '齐志奇', '赵欣', '刘力奇', '文爱国', '张炯', '张炯', '吴秀姣', '吴秀姣'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监', '财务总监'] ['2016-03-31', '2016-03-31', '2016-03-31', '2016-03-31', '2016-03-31', '2017-03-26', '2017-03-26', '2016-03-31', '2016-03-31']
1235 ['603589'] ['口子窖'] ['安徽口子酒业股份有限公司'] ['徐进', '段炼', '詹玉峰', '黄绍刚', '朱成寅', '徐钦祥', '范博'] ['总经理', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理'] ['2017-05-19', '2017-05-19', '2017-05-19', '2017-05-19', '2017-05-19', '2017-05-19', '2017-05-19']
1236 ['603590'] ['康辰药业'] ['北京康辰药业股份有限公司'] False False False
1237 ['603595'] ['东尼电子'] ['浙江东尼电子股份有限公司'] ['陈泉强', '沈晓宇', '丁勇', '李峰', '罗斌斌', '钟伟琴'] ['-', '总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2015-09-08', '2015-09-08', '2015-09-08', '2015-09-08', '2015-09-08', '2015-09-08']
1238 ['603596'] ['伯特利'] ['

1264 ['603639'] ['海利尔'] ['海利尔药业集团股份有限公司'] ['葛家成', '葛家成', '徐洪涛', '徐洪涛', '李建国', '杨波涛', '杨波涛', '汤安荣', '汤安荣', '方洁'] ['总经理', '总经理', '其他', '其他', '其他', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2017-08-28', '2017-08-28', '2017-08-28', '2017-08-28', '2017-08-28', '2017-08-28', '2017-08-28', '2017-08-28', '2017-08-28', '2018-04-25']
1265 ['603648'] ['畅联股份'] ['上海畅联国际物流股份有限公司'] ['周颖', '曲品南', '陆健', '茆英华', '沈侃', '施俊'] ['其他', '其他', '其他', '副总经理', '董事会秘书', '财务总监'] ['2017-04-01', '2017-04-01', '2017-04-01', '2017-04-01', '2017-04-01', '2017-04-01']
1266 ['603650'] ['彤程新材'] ['彤程新材料集团股份有限公司'] ['顾卫东', '徐捷'] ['-', '-'] ['2016-09-06', '2016-09-06']
1267 ['603655'] ['朗博科技'] ['常州朗博密封科技股份有限公司'] ['吴兴才', '潘建华'] ['-', '财务总监'] ['2015-10-26', '2015-10-26']
1268 ['603656'] ['泰禾光电'] ['合肥泰禾光电科技股份有限公司'] ['许大红', '许大红', '唐麟', '石江涛', '许梦生', '许梦生', '朱志虎', '石江涛', '唐麟', '黄慧丽', '黄慧丽'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '财务总监', '财务总监'] ['2015-06-08', '2018-06-08', '2015-06-08', '2018-06-08

1296 ['603703'] ['盛洋科技'] ['浙江盛洋科技股份有限公司'] ['叶建中', '徐凤娟', '吴秋婷', '叶利明', '张一鹏'] ['副总经理', '副总经理', '董事会秘书', '董事长', '财务总监'] ['2016-12-06', '2016-12-06', '2016-12-06', '2016-12-06', '2016-12-06']
1297 ['603706'] ['东方环宇'] ['新疆东方环宇燃气股份有限公司'] ['李伟伟'] ['董事会秘书'] ['2015-07-06']
1298 ['603707'] ['健友股份'] ['南京健友生化制药股份有限公司'] ['唐咏群', '吴桂萍', '黄锡伟'] ['总经理', '副总经理', '副总经理'] ['2017-05-12', '2017-05-12', '2017-05-12']
1299 ['603708'] ['家家悦'] ['家家悦集团股份有限公司'] ['丁明波', '郝芸', '丁明波', '李美', '郝芸', '周承生', '姜文霞', '姜文霞'] ['副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2016-05-16', '2016-05-16', '2016-05-16', '2018-04-10', '2016-05-16', '2018-04-10', '2016-05-16', '2016-05-16']
1300 ['603709'] ['中源家居'] ['中源家居股份有限公司'] ['张敏', '奚银'] ['副总经理', '董事会秘书'] ['2016-03-20', '2015-10-07']
1301 ['603711'] ['香飘飘'] ['香飘飘食品股份有限公司'] ['夏楠', '邹勇坚'] ['副总经理', '财务总监'] ['2018-06-26', '2018-04-26']
1302 ['603712'] ['七一二'] ['天津七一二通信广播股份有限公司'] ['庞辉', '马严', '张宝柱', '邓昌青', '马海永', '陈静'] ['总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财

1332 ['603798'] ['康普顿'] ['青岛康普顿科技股份有限公司'] ['焦广宇', '王强', '焦广宇', '王强', '王黎明', '张丽'] ['-', '-', '-', '-', '-', '副总经理'] ['2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23', '2017-05-23', '2018-05-10']
1333 ['603799'] ['华友钴业'] ['浙江华友钴业股份有限公司'] ['陈红良', '陈红良', '陈学', '徐伟', '徐伟', '王云', '方圆', '王云', '方圆', '陈要忠', '张炳海', '张炳海', '全昌胜', '李瑞', '李瑞'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2017-04-21', '2018-07-05', '2017-04-21', '2017-04-21']
1334 ['603800'] ['道森股份'] ['苏州道森钻采设备股份有限公司'] ['邹利明', '邹利明', '庄会涛', '裘伟斌', '李树林', '庄会涛', '李树林', '王骋', '王骋'] ['总经理', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28', '2018-03-28']
1335 ['603801'] ['志邦家居

1362 ['603861'] ['白云电器'] ['广州白云电器设备股份有限公司'] ['黄楚秋', '王卫彬', '王卫彬'] ['副总经理', '董事会秘书', '董事会秘书'] ['2018-04-26', '2016-12-29', '2016-12-29']
1363 ['603866'] ['桃李面包'] ['桃李面包股份有限公司'] ['吴学群', '吴学群', '吴学亮', '吴学亮', '张银安', '张银安', '谭秀坤', '谭秀坤'] ['总经理', '总经理', '其他', '其他', '董事会秘书', '董事会秘书', '财务总监', '财务总监'] ['2016-06-30', '2016-06-30', '2016-12-23', '2016-12-23', '2016-06-30', '2016-06-30', '2016-06-30', '2016-06-30']
1364 ['603868'] ['飞科电器'] ['上海飞科电器股份有限公司'] ['李丐腾', '李丐腾', '金文彩', '金文彩', '苏璠', '胡莹', '胡莹'] ['总经理', '总经理', '副总经理', '副总经理', '董事会秘书', '财务总监', '财务总监'] ['2012-11-09', '2012-11-09', '2012-11-09', '2012-11-09', '2018-01-23', '2014-03-09', '2014-03-09']
1365 ['603869'] ['新智认知'] ['北部湾旅游股份有限公司'] ['杨瑞', '刘德军', '王德惠', '朱晋宏', '郭树青', '杨瑞', '张建', '王东英', '檀国民', '王东英', '田芳', '田芳', '田芳'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '董事会秘书', '财务总监', '财务总监', '财务总监'] ['2017-06-21', '2013-12-23', '2015-08-27', '2015-07-06', '2015-07-06', '2017-01-01', '2018-02-08', '2017-02-28', '2

1392 ['603912'] ['佳力图'] ['南京佳力图机房环境技术股份有限公司'] ['王凌云', '郭端晓', '杜明伟', '袁祎', '李林达', '叶莉莉'] ['总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2015-10-08', '2015-10-08', '2015-10-08', '2015-10-08', '2015-10-08', '2015-10-08']
1393 ['603916'] ['苏博特'] ['江苏苏博特新材料股份有限公司'] ['冉千平', '洪锦祥', '陈建华', '张勇', '徐岳', '任红军'] ['其他', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2018-05-05', '2018-05-05', '2018-05-05', '2018-05-05', '2018-05-05', '2018-05-05']
1394 ['603917'] ['合力科技'] ['宁波合力模具科技股份有限公司'] ['施良才', '蔡振贤', '邬振贵', '蔡振权', '姚杰', '钱朝宝', '樊开曙', '樊开源', '曲翠红'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '财务总监'] ['2017-01-08', '2017-01-08', '2017-01-08', '2017-01-08', '2017-01-08', '2017-01-08', '2017-01-08', '2017-01-08', '2017-01-08']
1395 ['603918'] ['金桥信息'] ['上海金桥信息股份有限公司'] ['金史平', '王琨', '李志明'] ['副总经理', '副总经理', '董事会秘书'] ['2016-10-17', '2016-10-17', '2016-10-17']
1396 ['603919'] ['金徽酒'] ['金徽酒股份有限公司'] ['周志刚', '周志刚', '唐  云', '杜学义', '廖结兵', '王  栋', '张志刚', '张  斌', '杜学义', '张志刚', '王  栋', '廖

1424 ['603988'] ['中电电机'] ['中电电机股份有限公司'] ['王建裕', '刘国徽', '杨志明', '沈国新', '刘锴'] ['总经理', '副总经理', '副总经理', '副总经理', '董事会秘书'] ['2017-09-29', '2017-09-29', '2017-09-29', '2017-09-29', '2017-09-29']
1425 ['603989'] ['艾华集团'] ['湖南艾华集团股份有限公司'] ['王安安', '殷宝华', '艾亮', '陈晨', 'Stanley  Shoakan  Woo', '艾燕', '艾燕', '吴松青'] ['总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总监'] ['2015-11-16', '2015-11-16', '2015-11-16', '2018-06-21', '2018-06-21', '2017-02-22', '2017-02-22', '2015-11-16']
1426 ['603990'] ['麦迪科技'] ['苏州麦迪斯顿医疗科技股份有限公司'] ['汪建华', '傅洪', '陈泽江', '汪建华', '徐智渊', '徐智渊', '陈泽江', '傅洪', '孔烽', '孔烽'] ['-', '-', '-', '总经理', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书'] ['2015-08-01', '2015-08-01', '2015-08-01', '2015-08-01', '2015-08-01', '2015-08-01', '2015-08-01', '2015-08-01', '2015-08-01', '2015-08-01']
1427 ['603991'] ['至正股份'] ['上海至正道化高分子材料股份有限公司'] ['李现春', '宋刚', '宋刚', '项健', '宋静', '张天明', '宋静', '章玮琴', '章玮琴', '迪玲芳', '迪玲芳'] ['总经理', '其他', '其他', '副总经理', '副总经理', '副总经理', '副总经理', '董事会秘书', '董事会秘书', '财务总

{
    'company_1': {
        'stock_market': 'shanghai',
        'stock_code': '123456',
        'stock_name': '1234',
        'company_name': 'xxx',
        'senior_members': [
            {
                'name': 'abc',
                'occupation': 'ceo',
                'date_since': '2000-01-01',
            },
            # more ....
        ]
        # more ...
    },
    # more ...
}

In [735]:
def get_members(company):
    senior_list = []
    # if senior member is not null

    if(senior[c ompany]): 
        senior_num = len(senior[company])
        for i in range(senior_num):
            senior_mem = {
                'name' : senior[company][i],
                'occupation' : occupation[company][i],
                'data_since' : data_since[company][i]
            }
            senior_list.append(senior_mem)
    print(senior_list)
    return senior_list


In [736]:
#comp_code,comp_abbr,full_name,senior,occupation,data_since
def get_company():
    company_list= []
    company_num=len(comp_code)
    for i in range(company_num):
        senior_list = get_members(i)
        company_info = {
            'stock_market': 'shanghai',
            'stock_code': comp_code,
            'stock_name': comp_abbr,
            'company_name': full_name,
            'senior_members': senior_list
        }
        company_list.append(company_info)
    return company_list  

In [737]:
get_members(118)

[]


[]

In [738]:
company_list = get_company()

[{'name': '刘信义', 'occupation': '总经理', 'data_since': '2016-04-28'}, {'name': '王新浩', 'occupation': '副总经理', 'data_since': '2016-04-29'}, {'name': '崔炳文', 'occupation': '副总经理', 'data_since': '2016-04-29'}, {'name': '刘以研', 'occupation': '副总经理', 'data_since': '2016-04-29'}, {'name': '潘卫东', 'occupation': '副总经理', 'data_since': '2016-04-28'}, {'name': '崔炳文', 'occupation': '副总经理', 'data_since': '2016-04-29'}, {'name': '谢伟', 'occupation': '副总经理', 'data_since': '2016-04-29'}, {'name': '王新浩', 'occupation': '副总经理', 'data_since': '2016-04-29'}, {'name': '徐海燕', 'occupation': '副行长、党委委员', 'data_since': '2016-04-29'}, {'name': '徐海燕', 'occupation': '副行长、党委委员', 'data_since': '2016-04-29'}]
[{'name': '谢冰心', 'occupation': '总经济师', 'data_since': '2016-12-07'}, {'name': '王晓勇', 'occupation': '副总经理', 'data_since': '2016-08-30'}, {'name': '黄浩', 'occupation': '副总经理', 'data_since': '2016-12-07'}, {'name': '武宇', 'occupation': '副总经理', 'data_since': '2016-12-07'}, {'name': '王晓勇', 'occupation': '副总经理', 'data_since': '201

[{'name': '曹龙祥', 'occupation': '总经理', 'data_since': '2017-03-03'}, {'name': '史文正', 'occupation': '其他', 'data_since': '2017-03-03'}, {'name': '严宏泉', 'occupation': '副总经理', 'data_since': '2017-03-03'}, {'name': '张建民', 'occupation': '副总经理', 'data_since': '2017-03-03'}, {'name': '黄曲荣', 'occupation': '副总经理', 'data_since': '2017-03-03'}, {'name': '周其华', 'occupation': '副总经理', 'data_since': '2017-03-03'}, {'name': '吴宏亮', 'occupation': '财务总监', 'data_since': '2017-03-03'}]
[{'name': '吴明武', 'occupation': '总经理', 'data_since': '2017-11-30'}, {'name': '石春茂', 'occupation': '其他', 'data_since': '2017-11-30'}, {'name': '江玉林', 'occupation': '副总经理', 'data_since': '2017-11-30'}, {'name': '江玉林', 'occupation': '副总经理', 'data_since': '2017-11-30'}, {'name': '潘金堂', 'occupation': '副总经理', 'data_since': '2017-11-30'}, {'name': '孙晓民', 'occupation': '副总经理', 'data_since': '2017-11-30'}, {'name': '孙晓民', 'occupation': '副总经理', 'data_since': '2017-11-30'}, {'name': '杨昊悦', 'occupation': '董事会秘书', 'data_since': '2017-11-30'}

[{'name': '倪嘉宇', 'occupation': '-', 'data_since': '2017-08-23'}, {'name': '马刚', 'occupation': '-', 'data_since': '2017-08-23'}, {'name': '祁和刚', 'occupation': '-', 'data_since': '2006-08-18'}, {'name': '柴乔林', 'occupation': '总会计师', 'data_since': '2017-01-22'}, {'name': '柴乔林', 'occupation': '总会计师', 'data_since': '2017-01-22'}, {'name': '濮津', 'occupation': '副总裁', 'data_since': '2009-09-10'}, {'name': '周东洲', 'occupation': '董事会秘书兼联席公司秘书', 'data_since': '2006-08-18'}, {'name': '周东洲', 'occupation': '董事会秘书兼联席公司秘书', 'data_since': '2006-08-18'}]
[{'name': '蓝福生', 'occupation': '总经理', 'data_since': '2016-12-30'}, {'name': '郭先健', 'occupation': '其他', 'data_since': '2017-08-18'}, {'name': '范长文', 'occupation': '公司秘书', 'data_since': '2016-12-30'}, {'name': '范长文', 'occupation': '公司秘书', 'data_since': '2016-12-30'}, {'name': '方启学', 'occupation': '副总经理', 'data_since': '2016-12-30'}, {'name': '谢雄辉', 'occupation': '副总经理', 'data_since': '2016-12-30'}, {'name': '谢雄辉', 'occupation': '副总经理', 'data_since': '2016-1

[{'name': '倪兼明', 'occupation': '总经理', 'data_since': '2018-05-17'}, {'name': '何祚军', 'occupation': '副总经理', 'data_since': '2018-05-17'}, {'name': '连霞', 'occupation': '副总经理', 'data_since': '2018-05-17'}, {'name': '杨璐', 'occupation': '董事会秘书', 'data_since': '2018-05-17'}, {'name': '杨璐', 'occupation': '董事会秘书', 'data_since': '2018-05-17'}, {'name': '刘婉雯', 'occupation': '财务总监', 'data_since': '2018-05-17'}]
[{'name': '丁云峰', 'occupation': '总经理', 'data_since': '2018-02-02'}, {'name': '丁云峰', 'occupation': '总经理', 'data_since': '2018-02-02'}, {'name': '邵彩梅', 'occupation': '其他', 'data_since': '2018-02-02'}, {'name': '邵彩梅', 'occupation': '其他', 'data_since': '2018-02-02'}, {'name': '王凤久', 'occupation': '副总经理', 'data_since': '2018-02-02'}, {'name': '邱嘉辉', 'occupation': '副总经理', 'data_since': '2018-02-02'}, {'name': '王振勇', 'occupation': '副总经理', 'data_since': '2018-02-02'}, {'name': '王振勇', 'occupation': '副总经理', 'data_since': '2018-02-02'}, {'name': '王凤久', 'occupation': '副总经理', 'data_since': '2018-02-02'}, {

In [740]:
json_company = json.dumps(company_list) 

In [741]:
json_company

In [694]:
full_name[118]

['新疆亿路万源实业投资控股股份有限公司']